#
msgid ""
msgstr ""
"Project-Id-Version: notmusicbot\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-04 08:15-0800\n"
"PO-Revision-Date: 2024-11-06 14:31\n"
"Last-Translator: \n"
"Language-Team: Thai\n"
"Language: th_TH\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Crowdin-File: /fae/i18n/musicbot_logs.pot\n"
"X-Crowdin-File-ID: 26\n"
"X-Crowdin-Language: th\n"
"X-Crowdin-Project: notmusicbot\n"
"X-Crowdin-Project-ID: 734017\n"

#: musicbot/aliases.py:52
msgid "Aliases file not found, copying example_aliases.json"
msgstr ""

#: musicbot/aliases.py:80
#, python-format
msgid "Failed to load aliases file:  %s"
msgstr ""

#: musicbot/aliases.py:88
#, python-format
msgid ""
"Failed to parse aliases file:  %s\n"
"Ensure the file contains valid JSON and restart the bot."
msgstr ""

#: musicbot/aliases.py:118
#, python-format
msgid ""
"Aliases skipped for non-existent command:  %(command)s  ->  %(aliases)s"
msgstr ""

#: musicbot/aliases.py:126
#, python-format
msgid ""
"Alias(es) skipped for invalid alias data:  %(command)s  ->  %(aliases)s"
msgstr ""

#: musicbot/aliases.py:136
#, python-format
msgid "Alias `%(alias)s` skipped as already exists on command:  %(command)s"
msgstr ""

#: musicbot/autoplaylist.py:77
#, python-format
msgid "Error loading auto playlist file:  %s"
msgstr ""

#: musicbot/autoplaylist.py:181
#, python-format
msgid "Removing unplayable song from playlist, %(playlist)s: %(track)s"
msgstr ""

#: musicbot/autoplaylist.py:186
#, python-format
msgid "Removing song from playlist, %(playlist)s: %(track)s"
msgstr ""

#: musicbot/autoplaylist.py:132 musicbot/autoplaylist.py:211
msgid "Could not log information about the playlist URL removal."
msgstr ""

#: musicbot/autoplaylist.py:135 musicbot/autoplaylist.py:215
msgid "Updating playlist file..."
msgstr ""

#: musicbot/autoplaylist.py:158 musicbot/autoplaylist.py:231
#: musicbot/autoplaylist.py:268
#, python-format
msgid "Failed to save playlist file:  %s"
msgstr ""

#: musicbot/autoplaylist.py:240
#, python-format
msgid "URL already in playlist %s, ignoring"
msgstr ""

#: musicbot/autoplaylist.py:247
#, python-format
msgid "Adding new URL to playlist, %(playlist)s: %(track)s"
msgstr ""

#: musicbot/bot.py:166
#, python-format
msgid "Initializing MusicBot %s"
msgstr ""

#: musicbot/bot.py:249
#, python-format
msgid "Loop is closed, cannot create task for: %r"
msgstr ""

#: musicbot/bot.py:265
#, python-format
msgid "Unhandled exception for task:  %r"
msgstr ""

#: musicbot/bot.py:269
#, python-format
msgid "Unhandled exception for task:  %(task)r  --  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:314 musicbot/bot.py:336
msgid "Spotify did not provide us with a token. Disabling."
msgstr ""

#: musicbot/bot.py:318
msgid "Authenticated with Spotify successfully using client ID and secret."
msgstr ""

#: musicbot/bot.py:322
#, python-format
msgid ""
"Could not start Spotify client. Is your client ID and secret correct? "
"Details: %s. Continuing anyway in 5 seconds..."
msgstr ""

#: musicbot/bot.py:330
msgid ""
"The config did not have Spotify app credentials, attempting to use guest "
"mode."
msgstr ""

#: musicbot/bot.py:340
msgid "Authenticated with Spotify successfully using guest mode."
msgstr ""

#: musicbot/bot.py:345
#, python-format
msgid "Could not start Spotify client using guest mode. Details: %s."
msgstr ""

#: musicbot/bot.py:353
msgid ""
"Experimental Yt-dlp OAuth2 plugin is enabled. This might break at any point!"
msgstr ""

#: musicbot/bot.py:360
msgid "Initialized, now connecting to discord."
msgstr ""

#: musicbot/bot.py:371
msgid "Network ping test is disabled via config."
msgstr ""

#: musicbot/bot.py:375
msgid "Network ping test is closing down."
msgstr ""

#: musicbot/bot.py:386
msgid "Could not resolve ping target."
msgstr ""

#: musicbot/bot.py:416
msgid "Network ping test cancelled."
msgstr ""

#: musicbot/bot.py:430
msgid "Network testing via HTTP does not have a session to borrow."
msgstr ""

#: musicbot/bot.py:455
msgid "Could not locate `ping` executable in your environment."
msgstr ""

#: musicbot/bot.py:481
#, python-format
msgid ""
"MusicBot could not locate a `ping` command path.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr ""

#: musicbot/bot.py:491
#, python-format
msgid ""
"MusicBot was denied permission to execute the `ping` command.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr ""

#: musicbot/bot.py:501
#, python-format
msgid ""
"Your environment may not allow the `ping` system command.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr ""

#: musicbot/bot.py:516
msgid "MusicBot detected network is available again."
msgstr ""

#: musicbot/bot.py:521
msgid "VoiceClient is not connected, waiting to resume MusicPlayer..."
msgstr ""

#: musicbot/bot.py:525
#, python-format
msgid "Resuming playback of player:  (%(guild_id)s) %(player)r"
msgstr ""

#: musicbot/bot.py:536
msgid "MusicBot detected a network outage."
msgstr ""

#: musicbot/bot.py:540
#, python-format
msgid ""
"Pausing MusicPlayer due to network availability:  (%(guild_id)s) %(player)r"
msgstr ""

#: musicbot/bot.py:561
#, python-format
msgid ""
"Looking for owner in guild: %(guild)s (required voice: %(required)s) and "
"got:  %(owner)s"
msgstr ""

#: musicbot/bot.py:575
msgid "Checking for channels to auto-join or resume..."
msgstr ""

#: musicbot/bot.py:587
#, python-format
msgid "Guild not available, cannot auto join:  %(id)s/%(name)s"
msgstr ""

#: musicbot/bot.py:595
#, python-format
msgid "Found resumable voice channel:  %(channel)s  in guild:  %(guild)s"
msgstr ""

#: musicbot/bot.py:608
#, python-format
msgid "Will try resuming voice session instead of Auto-Joining channel:  %s"
msgstr ""

#: musicbot/bot.py:625
#, python-format
msgid "Found owner in voice channel:  %s"
msgstr ""

#: musicbot/bot.py:630
#, python-format
msgid "Ignoring resumable channel, AutoSummon to owner in channel:  %s"
msgstr ""

#: musicbot/bot.py:635
#, python-format
msgid "Ignoring Auto-Join channel, AutoSummon to owner in channel:  %s"
msgstr ""

#: musicbot/bot.py:647
#, python-format
msgid "Already connected to channel:  %(channel)s  in guild:  %(guild)s"
msgstr ""

#: musicbot/bot.py:656
#, python-format
msgid "Attempting to join channel:  %(channel)s  in guild:  %(guild)s"
msgstr ""

#: musicbot/bot.py:663
msgid "Discarding MusicPlayer and making a new one..."
msgstr ""

#: musicbot/bot.py:683
msgid "MusicBot will make a new MusicPlayer now..."
msgstr ""

#: musicbot/bot.py:704
#, python-format
msgid "Not joining %(guild)s/%(channel)s, it isn't a supported voice channel."
msgstr ""

#: musicbot/bot.py:707
msgid "Finished joining configured channels."
msgstr ""

#: musicbot/bot.py:730
msgid "Member is not voice-enabled and cannot use this command."
msgstr ""

#: musicbot/bot.py:738
msgid "You cannot use this command when not in the voice channel."
msgstr ""

#: musicbot/bot.py:752
msgid "Getting bot Application Info."
msgstr ""

#: musicbot/bot.py:777
#, python-format
msgid "MusicBot does not have permission to Connect in channel:  %s"
msgstr ""

#: musicbot/bot.py:781
#, python-format
msgid "MusicBot does not have permission to Connect in channel:  `%(name)s`"
msgstr ""

#: musicbot/bot.py:786
#, python-format
msgid "MusicBot does not have permission to Speak in channel:  %s"
msgstr ""

#: musicbot/bot.py:790
#, python-format
msgid "MusicBot does not have permission to Speak in channel:  `%(name)s`"
msgstr ""

#: musicbot/bot.py:800
#, python-format
msgid "Reusing bots VoiceClient from guild:  %s"
msgstr ""

#: musicbot/bot.py:805
#, python-format
msgid "Forcing disconnect on stale VoiceClient in guild:  %s"
msgstr ""

#: musicbot/bot.py:811
msgid "Disconnect failed or was cancelled?"
msgstr ""

#: musicbot/bot.py:819
#, python-format
msgid "MusicBot is unable to connect to the channel right now:  %(channel)s"
msgstr ""

#: musicbot/bot.py:823
msgid ""
"MusicBot could not connect to the channel.\n"
"Try again later, or restart the bot if this continues."
msgstr ""

#: musicbot/bot.py:834
msgid "MusicBot has a VoiceClient now..."
msgstr ""

#: musicbot/bot.py:839
#, python-format
msgid ""
"Retrying connection after a timeout error (%(attempt)s) while trying to "
"connect to:  %(channel)s"
msgstr ""

#: musicbot/bot.py:844
msgid "MusicBot VoiceClient connection attempt was cancelled. No retry."
msgstr ""

#: musicbot/bot.py:847
msgid ""
"MusicBot connection to voice was cancelled. This is odd. Maybe restart?"
msgstr ""

#: musicbot/bot.py:858 musicbot/bot.py:8598
msgid "MusicBot does not have permission to speak."
msgstr ""

#: musicbot/bot.py:862 musicbot/bot.py:8600
msgid "MusicBot could not request to speak."
msgstr ""

#: musicbot/bot.py:875
#, python-format
msgid "Disconnecting a MusicPlayer in guild:  %s"
msgstr ""

#: musicbot/bot.py:887
msgid "Disconnecting VoiceClient before we kill the MusicPlayer."
msgstr ""

#: musicbot/bot.py:895 musicbot/bot.py:914 musicbot/bot.py:926
#: musicbot/bot.py:949
msgid "The disconnect failed or was cancelled."
msgstr ""

#: musicbot/bot.py:905
msgid ""
"MusicBot has a VoiceProtocol that is not a VoiceClient. Disconnecting "
"anyway..."
msgstr ""

#: musicbot/bot.py:918
#, python-format
msgid "Disconnecting a rogue VoiceClient in guild:  %s"
msgstr ""

#: musicbot/bot.py:942
msgid "Disconnecting a non-guild VoiceClient..."
msgstr ""

#: musicbot/bot.py:952
#, python-format
msgid ""
"MusicBot.voice_clients list contains a non-VoiceClient object?\n"
"The object is actually of type:  %s"
msgstr ""

#: musicbot/bot.py:962
#, python-format
msgid "We still have a MusicPlayer ref in guild (%(guild_id)s):  %(player)r"
msgstr ""

#: musicbot/bot.py:975
#, python-format
msgid "Guild (%(guild)s) wants a player, optional:  %(player)r"
msgstr ""

#: musicbot/bot.py:987
msgid "MusicPlayer has a VoiceClient that is not connected."
msgstr ""

#: musicbot/bot.py:988
#, python-format
msgid "MusicPlayer obj:  %r"
msgstr ""

#: musicbot/bot.py:989
#, python-format
msgid "VoiceClient obj:  %r"
msgstr ""

#: musicbot/bot.py:1009
#, python-format
msgid ""
"Getting a MusicPlayer for guild:  %(guild)s  In Channel:  %(channel)s  "
"Create: %(create)s  Deserialize:  %(serial)s"
msgstr ""

#: musicbot/bot.py:1025
#, python-format
msgid ""
"Created player via deserialization for guild %(guild_id)s with %(number)s "
"entries"
msgstr ""

#: musicbot/bot.py:1034
msgid ""
"The bot is not in a voice channel.\n"
"Use the summon command to bring the bot to your voice channel."
msgstr ""

#: musicbot/bot.py:1046
msgid "Something is wrong, we didn't get the VoiceClient."
msgstr ""

#: musicbot/bot.py:1079
msgid "Running on_player_play"
msgstr ""

#: musicbot/bot.py:1154
#, python-format
msgid "Setting URL history guild %(guild_id)s == %(url)s"
msgstr ""

#: musicbot/bot.py:1202
msgid "no channel to put now playing message into"
msgstr ""

#: musicbot/bot.py:1214
msgid "ignored now-playing message as it was already posted."
msgstr ""

#: musicbot/bot.py:1238
msgid "Running on_player_resume"
msgstr ""

#: musicbot/bot.py:1251
msgid "Running on_player_pause"
msgstr ""

#: musicbot/bot.py:1267
msgid "Running on_player_stop"
msgstr ""

#: musicbot/bot.py:1277
msgid "Running on_player_finished_playing"
msgstr ""

#: musicbot/bot.py:1279 musicbot/bot.py:1325 musicbot/bot.py:1396
msgid "Event loop is closed, nothing else to do here."
msgstr ""

#: musicbot/bot.py:1283 musicbot/bot.py:1329 musicbot/bot.py:1400
msgid "Logout under way, ignoring this event."
msgstr ""

#: musicbot/bot.py:1300
msgid "VoiceClient says it is not connected, nothing else we can do here."
msgstr ""

#: musicbot/bot.py:1307
msgid "Player finished and queue is empty, leaving voice channel..."
msgstr ""

#: musicbot/bot.py:1321
msgid "Looping over queue to expunge songs with missing author..."
msgstr ""

#: musicbot/bot.py:1360
#, python-format
msgid ""
"Author `%(user)s` absent, skipped (deleted) entry from queue:  %(song)s"
msgstr ""

#: musicbot/bot.py:1379
msgid "No playable songs in the Guild autoplaylist, disabling."
msgstr ""

#: musicbot/bot.py:1384
msgid "No content in current autoplaylist. Filling with new music..."
msgstr ""

#: musicbot/bot.py:1392
msgid "Looping over player autoplaylist..."
msgstr ""

#: musicbot/bot.py:1433
#, python-format
msgid "Error while processing song \"%(url)s\":  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:1444
#, python-format
msgid "Error extracting song \"%(url)s\": %(raw_error)s"
msgstr ""

#: musicbot/bot.py:1464
msgid "MusicBot got an unhandled exception in player finished event."
msgstr ""

#: musicbot/bot.py:1470
#, python-format
msgid "Expanding auto playlist with entries extracted from:  %s"
msgstr ""

#: musicbot/bot.py:1494
#, python-format
msgid "Error adding song from autoplaylist: %s"
msgstr ""

#: musicbot/bot.py:1497
msgid "Exception data for above error:"
msgstr ""

#: musicbot/bot.py:1503
msgid "No playable songs in the autoplaylist, disabling."
msgstr ""

#: musicbot/bot.py:1523
msgid "Running on_player_entry_added"
msgstr ""

#: musicbot/bot.py:1533
msgid "Automatically skipping auto-playlist entry for queued entry."
msgstr ""

#: musicbot/bot.py:1553
#, python-format
msgid "MusicPlayer exception for entry: %r"
msgstr ""

#: musicbot/bot.py:1559
msgid "MusicPlayer exception."
msgstr ""

#: musicbot/bot.py:1583
#, python-format
msgid "Auto playlist track could not be played:  %r"
msgstr ""

#: musicbot/bot.py:1622
msgid "Logout under way, ignoring status update event."
msgstr ""

#: musicbot/bot.py:1723
#, python-format
msgid "Update bot status:  %(status)s -- %(activity)r"
msgstr ""

#: musicbot/bot.py:1751
#, python-format
msgid "Serializing queue for %s"
msgstr ""

#: musicbot/bot.py:1777
#, python-format
msgid "Deserializing queue for %s"
msgstr ""

#: musicbot/bot.py:1795
#, python-format
msgid "Writing current song for %s"
msgstr ""

#: musicbot/bot.py:1825
msgid "[Dev Bug] Tried sending an invalid response object."
msgstr ""

#: musicbot/bot.py:1853
#, python-format
msgid "sending embed to: %s"
msgstr ""

#: musicbot/bot.py:1856
#, python-format
msgid "sending text to: %s"
msgstr ""

#: musicbot/bot.py:1861
#, python-format
msgid "Cannot send message to \"%s\", no permission"
msgstr ""

#: musicbot/bot.py:1868
#, python-format
msgid "Cannot send message to \"%s\", invalid or deleted channel"
msgstr ""

#: musicbot/bot.py:1876
#, python-format
msgid "Message is over the message size limit (%s)"
msgstr ""

#: musicbot/bot.py:1884
msgid "Could not send private message, sending in fallback channel instead."
msgstr ""

#: musicbot/bot.py:1902
#, python-format
msgid "Rate limited send message, retrying in %s seconds."
msgstr ""

#: musicbot/bot.py:1908
#, python-format
msgid "Cancelled message retry for:  %s"
msgstr ""

#: musicbot/bot.py:1913
msgid "Rate limited send message, but cannot retry!"
msgstr ""

#: musicbot/bot.py:1919
msgid "Failed to send message in fallback channel."
msgstr ""

#: musicbot/bot.py:1924 musicbot/bot.py:1988 musicbot/bot.py:2058
msgid "Failed to send due to an HTTP error."
msgstr ""

#: musicbot/bot.py:1950
#, python-format
msgid "Cannot delete message \"%s\", no permission"
msgstr ""

#: musicbot/bot.py:1955
#, python-format
msgid "Cannot delete message \"%s\", message not found"
msgstr ""

#: musicbot/bot.py:1973
#, python-format
msgid "Rate limited message delete, retrying in %s seconds."
msgstr ""

#: musicbot/bot.py:1978
msgid "Rate limited message delete, but cannot retry!"
msgstr ""

#: musicbot/bot.py:1981
msgid "Failed to delete message"
msgstr ""

#: musicbot/bot.py:1983
#, python-format
msgid "Got HTTPException trying to delete message: %s"
msgstr ""

#: musicbot/bot.py:2018
#, python-format
msgid "Cannot edit message \"%s\", message not found"
msgstr ""

#: musicbot/bot.py:2022
msgid "Sending message instead"
msgstr ""

#: musicbot/bot.py:2039
#, python-format
msgid "Rate limited edit message, retrying in %s seconds."
msgstr ""

#: musicbot/bot.py:2045
#, python-format
msgid "Cancelled message edit for:  %s"
msgstr ""

#: musicbot/bot.py:2051
msgid "Failed to edit message"
msgstr ""

#: musicbot/bot.py:2053
#, python-format
msgid "Got HTTPException trying to edit message %s to: %s"
msgstr ""

#: musicbot/bot.py:2075
#, python-format
msgid "Cancelled delete for message (ID: %(id)s):  %(content)s"
msgstr ""

#: musicbot/bot.py:2137
#, python-format
msgid "Caught a signal from the OS: %s"
msgstr ""

#: musicbot/bot.py:2141
msgid "Disconnecting and closing down MusicBot..."
msgstr ""

#: musicbot/bot.py:2144
msgid "Exception thrown while handling interrupt signal!"
msgstr ""

#: musicbot/bot.py:2159
msgid "MusicBot is now doing shutdown steps..."
msgstr ""

#: musicbot/bot.py:2167
msgid ""
"Failed Discord API Login!\n"
"\n"
"Problem:\n"
"  MusicBot could not log into Discord API.\n"
"  Your Token may be incorrect or there may be an API outage.\n"
"\n"
"Solution:\n"
"  Make sure you have the correct Token set in your config.\n"
"  Check API status at the official site: discordstatus.com"
msgstr ""

#: musicbot/bot.py:2181
msgid "Waiting for download threads to finish up..."
msgstr ""

#: musicbot/bot.py:2207
#, python-format
msgid "Will wait for task:  %(name)s  (%(func)s)"
msgstr ""

#: musicbot/bot.py:2214
#, python-format
msgid "Will try to cancel task:  %(name)s  (%(func)s)"
msgstr ""

#: musicbot/bot.py:2222
msgid "Awaiting pending tasks..."
msgstr ""

#: musicbot/bot.py:2228
msgid "Closing HTTP Connector."
msgstr ""

#: musicbot/bot.py:2234
msgid "Closing aiohttp session."
msgstr ""

#: musicbot/bot.py:2246
msgid "Logout has been called."
msgstr ""

#: musicbot/bot.py:2258
#, python-format
msgid ""
"Exception in %(event)s:\n"
"%(error)s"
msgstr ""

#: musicbot/bot.py:2274
#, python-format
msgid "Exception in %s"
msgstr ""

#: musicbot/bot.py:2281
msgid "MusicBot resumed a session with discord."
msgstr ""

#: musicbot/bot.py:2298
msgid "Finish on_ready"
msgstr ""

#: musicbot/bot.py:2305
msgid "Logged in, now getting MusicBot ready..."
msgstr ""

#: musicbot/bot.py:2308
msgid "ClientUser is somehow none, we gotta bail..."
msgstr ""

#: musicbot/bot.py:2317
#, python-format
msgid "MusicBot:  %(id)s/%(name)s#%(desc)s"
msgstr ""

#: musicbot/bot.py:2328
#, python-format
msgid "Owner:     %(id)s/%(name)s#%(desc)s\n"
msgstr ""

#: musicbot/bot.py:2336 musicbot/bot.py:2363
msgid "Guild List:"
msgstr ""

#: musicbot/bot.py:2340 musicbot/bot.py:2366 musicbot/bot.py:8784
#, python-format
msgid " - %s"
msgstr ""

#: musicbot/bot.py:2349
#, python-format
msgid "Left %s due to bot owner not found"
msgstr ""

#: musicbot/bot.py:2354
#, python-format
msgid "Not proceeding with checks in %s servers due to unavailability"
msgstr ""

#: musicbot/bot.py:2360
#, python-format
msgid "Owner could not be found on any guild (id: %s)\n"
msgstr ""

#: musicbot/bot.py:2369
msgid "Owner unknown, bot is not on any guilds."
msgstr ""

#: musicbot/bot.py:2373
#, python-format
msgid ""
"To make the bot join a guild, paste this link in your browser. \n"
"Note: You should be logged into your main account and have \n"
"manage server permissions on the guild you want the bot to join.\n"
"  %s"
msgstr ""

#: musicbot/bot.py:2390
#, python-format
msgid "Got None for bound channel with ID:  %d"
msgstr ""

#: musicbot/bot.py:2410
msgid "Bound to text channels:"
msgstr ""

#: musicbot/bot.py:2428 musicbot/bot.py:2478
#, python-format
msgid " - %(guild)s/%(channel)s"
msgstr ""

#: musicbot/bot.py:2432 musicbot/bot.py:2434
msgid "Not bound to any text channels"
msgstr ""

#: musicbot/bot.py:2499
#, python-format
msgid ""
"Detected missing config options!\n"
"\n"
"Problem:\n"
"  You config options file is missing some options.\n"
"  Default settings will be used for these options.\n"
"  Here is a list of options we didn't find:\n"
"  %(missing)s\n"
"\n"
"Solution:\n"
"  Copy new options from the example options file.\n"
"  Or use the config command to set and save them.\n"
"\n"
msgstr ""

#: musicbot/bot.py:2528
#, python-format
msgid "Event on_ready has fired %s times"
msgstr ""

#: musicbot/bot.py:2544
msgid "Getting application info."
msgstr ""

#: musicbot/bot.py:2562
msgid "Ensuring data folders exist"
msgstr ""

#: musicbot/bot.py:2577
msgid "Validating config"
msgstr ""

#: musicbot/bot.py:2580
msgid "Validating permissions config"
msgstr ""

#: musicbot/bot.py:2590
msgid "Disabled"
msgstr ""

#: musicbot/bot.py:2590
msgid "Enabled"
msgstr ""

#: musicbot/bot.py:2593
msgid "Options:"
msgstr ""

#: musicbot/bot.py:2595
#, python-format
msgid "  Command prefix: %s"
msgstr ""

#: musicbot/bot.py:2596
#, python-format
msgid "  Default volume: %d%%"
msgstr ""

#: musicbot/bot.py:2598
#, python-format
msgid "  Skip threshold: %(num)d votes or %(percent).0f%%"
msgstr ""

#: musicbot/bot.py:2605
#, python-format
msgid "  Now Playing @mentions: %s"
msgstr ""

#: musicbot/bot.py:2608
#, python-format
msgid "  Auto-Summon: %s"
msgstr ""

#: musicbot/bot.py:2610
#, python-format
msgid "  Auto-Playlist: %(status)s (order: %(order)s)"
msgstr ""

#: musicbot/bot.py:2613
msgid "random"
msgstr ""

#: musicbot/bot.py:2613
msgid "sequential"
msgstr ""

#: musicbot/bot.py:2618
#, python-format
msgid "  Auto-Pause: %s"
msgstr ""

#: musicbot/bot.py:2620
#, python-format
msgid "  Delete Messages: %s"
msgstr ""

#: musicbot/bot.py:2625
#, python-format
msgid "    Delete Invoking: %s"
msgstr ""

#: musicbot/bot.py:2632
#, python-format
msgid "  Debug Mode: %s"
msgstr ""

#: musicbot/bot.py:2634
#, python-format
msgid "  Downloaded songs will be %s"
msgstr ""

#: musicbot/bot.py:2638
#, python-format
msgid "    Delete if unused for %d days"
msgstr ""

#: musicbot/bot.py:2641
#, python-format
msgid "    Delete if size exceeds %s"
msgstr ""

#: musicbot/bot.py:2644
#, python-format
msgid "  Status message: %s"
msgstr ""

#: musicbot/bot.py:2646
#, python-format
msgid "  Write current songs to file: %s"
msgstr ""

#: musicbot/bot.py:2650
#, python-format
msgid "  Author insta-skip: %s"
msgstr ""

#: musicbot/bot.py:2653
#, python-format
msgid "  Embeds: %s"
msgstr ""

#: musicbot/bot.py:2655
#, python-format
msgid "  Spotify integration: %s"
msgstr ""

#: musicbot/bot.py:2658
#, python-format
msgid "  Legacy skip: %s"
msgstr ""

#: musicbot/bot.py:2660
#, python-format
msgid "  Leave non owners: %s"
msgstr ""

#: musicbot/bot.py:2664
#, python-format
msgid "  Leave inactive VC: %s"
msgstr ""

#: musicbot/bot.py:2669
#, python-format
msgid "    Timeout: %s seconds"
msgstr ""

#: musicbot/bot.py:2673
#, python-format
msgid "  Leave at song end/empty queue: %s"
msgstr ""

#: musicbot/bot.py:2677
#, python-format
msgid "  Leave when player idles: %s"
msgstr ""

#: musicbot/bot.py:2681
#, python-format
msgid "    Timeout: %d seconds"
msgstr ""

#: musicbot/bot.py:2682
#, python-format
msgid "  Self Deafen: %s"
msgstr ""

#: musicbot/bot.py:2684
#, python-format
msgid "  Per-server command prefix: %s"
msgstr ""

#: musicbot/bot.py:2687
#, python-format
msgid "  Search List: %s"
msgstr ""

#: musicbot/bot.py:2689
#, python-format
msgid "  Round Robin Queue: %s"
msgstr ""

#: musicbot/bot.py:2739
msgid ""
"Attempted to handle Voice Channel inactivity, but Bot is not in voice..."
msgstr ""

#: musicbot/bot.py:2746
#, python-format
msgid "Channel activity already waiting in guild: %s"
msgstr ""

#: musicbot/bot.py:2756
#, python-format
msgid ""
"Channel activity waiting %(time)d seconds to leave channel: %(channel)s"
msgstr ""

#: musicbot/bot.py:2771
#, python-format
msgid "Channel activity timer for %s has expired. Disconnecting."
msgstr ""

#: musicbot/bot.py:2777
#, python-format
msgid "Channel activity timer canceled for: %(channel)s in %(guild)s"
msgstr ""

#: musicbot/bot.py:2805
#, python-format
msgid "Ignoring player inactivity in auto-joined channel:  %s"
msgstr ""

#: musicbot/bot.py:2812
#, python-format
msgid "Player activity timer already waiting in guild: %s"
msgstr ""

#: musicbot/bot.py:2820
#, python-format
msgid ""
"Player activity timer waiting %(time)d seconds to leave channel: %(channel)s"
msgstr ""

#: musicbot/bot.py:2832
#, python-format
msgid "Player activity timer for %s has expired. Disconnecting."
msgstr ""

#: musicbot/bot.py:2838 musicbot/bot.py:2843
#, python-format
msgid "Player activity timer canceled for: %(channel)s in %(guild)s"
msgstr ""

#: musicbot/bot.py:2860
msgid "Player activity timer is being reset."
msgstr ""

#: musicbot/bot.py:2969
msgid "No such command"
msgstr ""

#: musicbot/bot.py:3045
msgid "You must mention a user or provide their ID number."
msgstr ""

#: musicbot/bot.py:3050
msgid "Invalid sub-command given. Use `help blockuser` for usage examples."
msgstr ""

#: musicbot/bot.py:3061
msgid "MusicBot could not find the user(s) you specified."
msgstr ""

#: musicbot/bot.py:3068
msgid "The owner cannot be added to the block list."
msgstr ""

#: musicbot/bot.py:3072
#, python-format
msgid "Not adding user to block list, already blocked:  %(id)s/%(name)s"
msgstr ""

#: musicbot/bot.py:3098
msgid "Cannot add the users you listed, they are already added."
msgstr ""

#: musicbot/bot.py:3175
msgid "You must provide a song subject if no song is currently playing."
msgstr ""

#: musicbot/bot.py:3193
#, python-format
msgid "Subject `%(subject)s` is already in the song block list."
msgstr ""

#: musicbot/bot.py:3225
msgid "The subject is not in the song block list and cannot be removed."
msgstr ""

#: musicbot/bot.py:3297
msgid "Invalid sub-command given. Use `help autoplaylist` for usage examples."
msgstr ""

#: musicbot/bot.py:3304
msgid "The supplied song link is invalid"
msgstr ""

#: musicbot/bot.py:3310
msgid "The queue is empty. Add some songs with a play command!"
msgstr ""

#: musicbot/bot.py:3338
msgid "This song is already in the autoplaylist."
msgstr ""

#: musicbot/bot.py:3355
msgid "This song is not yet in the autoplaylist."
msgstr ""

#: musicbot/bot.py:3389
msgid "You must provide a playlist filename."
msgstr ""

#: musicbot/bot.py:3499
msgid "You are not allowed to request playlists"
msgstr ""

#: musicbot/bot.py:3507
#, python-format
msgid "Playlist has too many entries (%(songs)s but max is %(max)s)"
msgstr ""

#: musicbot/bot.py:3518
#, python-format
msgid ""
"The playlist entries will exceed your queue limit.\n"
"There are %(songs)s in the list, and %(queued)s already in queue.\n"
"The limit is %(max)s for your group."
msgstr ""

#: musicbot/bot.py:3542
msgid "Ignoring auto-pause due to network outage."
msgstr ""

#: musicbot/bot.py:3547
msgid ""
"MusicPlayer has no VoiceClient or has no channel data, cannot process auto-"
"pause."
msgstr ""

#: musicbot/bot.py:3558
msgid "Already processing auto-pause, ignoring this event."
msgstr ""

#: musicbot/bot.py:3566
#, python-format
msgid ""
"%sVoiceClient not connected, waiting %s seconds to handle auto-pause in "
"guild:  %s"
msgstr ""

#: musicbot/bot.py:3574
msgid "Auto-pause waiting was cancelled."
msgstr ""

#: musicbot/bot.py:3581
msgid "A new MusicPlayer is being connected, ignoring old auto-pause event."
msgstr ""

#: musicbot/bot.py:3597
#, python-format
msgid "Playing in an empty voice channel, running auto pause for guild: %s"
msgstr ""

#: musicbot/bot.py:3604
#, python-format
msgid "Previously auto paused player is unpausing for guild: %s"
msgstr ""

#: musicbot/bot.py:3835
msgid "Cannot use seek if there is nothing playing."
msgstr ""

#: musicbot/bot.py:3840
msgid "Cannot use seek on current track, it has an unknown duration."
msgstr ""

#: musicbot/bot.py:3846
msgid "Seeking is not supported for streams."
msgstr ""

#: musicbot/bot.py:3856
msgid "Cannot use seek without a time to position playback."
msgstr ""

#: musicbot/bot.py:3874
#, python-format
msgid "Could not convert `%(input)s` to a valid time in seconds."
msgstr ""

#: musicbot/bot.py:3887
#, python-format
msgid ""
"Cannot seek to `%(input)s` (`%(seconds)s` seconds) in the current track with"
" a length of `%(progress)s / %(total)s`"
msgstr ""

#: musicbot/bot.py:3948
msgid "Invalid sub-command. Use the command `help repeat` for usage examples."
msgstr ""

#: musicbot/bot.py:3986
msgid "The player is not currently looping."
msgstr ""

#: musicbot/bot.py:4042
msgid "Song positions must be integers!"
msgstr ""

#: musicbot/bot.py:4047
msgid "You gave a position outside the playlist size!"
msgstr ""

#: musicbot/bot.py:4095
msgid ""
"Could not prompt for playlist playback, no message to add reactions to."
msgstr ""

#: musicbot/bot.py:4184
msgid "Local media playback is not enabled."
msgstr ""

#: musicbot/bot.py:4225
msgid "Spotify URL is invalid or not currently supported."
msgstr ""

#: musicbot/bot.py:4239 musicbot/bot.py:4444
#, python-format
msgid "You have reached your enqueued song limit (%(max)s)"
msgstr ""

#: musicbot/bot.py:4245 musicbot/bot.py:4450 musicbot/bot.py:4544
msgid "Karaoke mode is enabled, please try again when its disabled!"
msgstr ""

#: musicbot/bot.py:4257
msgid "Issue with extract_info(): "
msgstr ""

#: musicbot/bot.py:4261 musicbot/bot.py:4466
#, python-format
msgid ""
"Failed to extract info due to error:\n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:4267
msgid "That video cannot be played. Try using the stream command."
msgstr ""

#: musicbot/bot.py:4308
#, python-format
msgid ""
"Processed %(number)d of %(total)d songs in %(time).3f seconds at "
"%(time_per).2f s/song"
msgstr ""

#: musicbot/bot.py:4319
#, python-format
msgid ""
"No songs were added, all songs were over max duration (%(max)s seconds)"
msgstr ""

#: musicbot/bot.py:4347
#, python-format
msgid "Song duration exceeds limit (%(length)s > %(max)s)"
msgstr ""

#: musicbot/bot.py:4365
#, python-format
msgid "Added song(s) at position %s"
msgstr ""

#: musicbot/bot.py:4402
#, python-format
msgid "Cannot estimate time until playing for position: %d"
msgstr ""

#: musicbot/bot.py:4463
#, python-format
msgid "Failed to get info from the stream request: %s"
msgstr ""

#: musicbot/bot.py:4472
msgid "Streaming playlists is not yet supported."
msgstr ""

#: musicbot/bot.py:4538
#, python-format
msgid "You have reached your playlist item limit (%(max)s)"
msgstr ""

#: musicbot/bot.py:4550
msgid ""
"Please specify a search query.  Use `help search` for more information."
msgstr ""

#: musicbot/bot.py:4579
#, python-format
msgid "You cannot search for more than %(max)s videos"
msgstr ""

#: musicbot/bot.py:4904
#, python-format
msgid "Waiting for summon lock: %s"
msgstr ""

#: musicbot/bot.py:4907
#, python-format
msgid "Summon lock acquired for: %s"
msgstr ""

#: musicbot/bot.py:4911
msgid "You are not connected to voice. Try joining a voice channel!"
msgstr ""

#: musicbot/bot.py:4936
#, python-format
msgid "Joining %(guild)s/%(channel)s"
msgstr ""

#: musicbot/bot.py:4998
msgid "MusicBot cannot follow a user that is not a member of the server."
msgstr ""

#: musicbot/bot.py:5026
msgid "Player is not playing."
msgstr ""

#: musicbot/bot.py:5151
msgid "Nothing in the queue to remove!"
msgstr ""

#: musicbot/bot.py:5172
#, python-format
msgid "Nothing found in the queue from user `%(user)s`"
msgstr ""

#: musicbot/bot.py:5177 musicbot/bot.py:5213
msgid ""
"You do not have the permission to remove that entry from the queue.\n"
"You must be the one who queued it or have instant skip permissions."
msgstr ""

#: musicbot/bot.py:5188 musicbot/bot.py:5193
msgid "Invalid entry number. Use the queue command to find queue positions."
msgstr ""

#: musicbot/bot.py:5244
msgid "Can't skip! The player is not playing!"
msgstr ""

#: musicbot/bot.py:5296
msgid "You do not have permission to force skip a looped song."
msgstr ""

#: musicbot/bot.py:5316
msgid "You do not have permission to force skip."
msgstr ""

#: musicbot/bot.py:5353 musicbot/bot.py:5386
msgid "You do not have permission to skip a looped song."
msgstr ""

#: musicbot/bot.py:5436
#, python-format
msgid "`%(new_volume)s` is not a valid number"
msgstr ""

#: musicbot/bot.py:5477
#, python-format
msgid ""
"Unreasonable volume change provided: %(old_volume)s%(adjustment)s is %(new_volume)s.\n"
"Volume can only be set from 1 to 100."
msgstr ""

#: musicbot/bot.py:5487
#, python-format
msgid ""
"Unreasonable volume provided: %(volume)s. Provide a value between 1 and 100."
msgstr ""

#: musicbot/bot.py:5511
msgid ""
"No track is playing, cannot set speed.\n"
"Use the config command to set a default playback speed."
msgstr ""

#: musicbot/bot.py:5519
msgid "Speed cannot be applied to streamed media."
msgstr ""

#: musicbot/bot.py:5524
msgid "You must provide a speed to set."
msgstr ""

#: musicbot/bot.py:5592 musicbot/bot.py:5708 musicbot/bot.py:6638
#, python-format
msgid "Invalid option for command: `%(option)s`"
msgstr ""

#: musicbot/bot.py:5606
#, python-format
msgid ""
"Failed to save aliases due to error:\n"
"`%(raw_error)s`"
msgstr ""

#: musicbot/bot.py:5613
msgid "You must supply an alias and a command to alias"
msgstr ""

#: musicbot/bot.py:5628
msgid "You must supply an alias name to remove."
msgstr ""

#: musicbot/bot.py:5633
#, python-format
msgid "The alias `%(alias)s` does not exist."
msgstr ""

#: musicbot/bot.py:5691
msgid "Config cannot use channel and user mentions at the same time."
msgstr ""

#: musicbot/bot.py:5794
#, python-format
msgid ""
"Unable to reload Config due to the following error:\n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:5812
msgid ""
"Could not resolve section name from option name. Please provide a valid "
"section and option name."
msgstr ""

#: musicbot/bot.py:5816
msgid "The option given is ambiguous, please provide a section name."
msgstr ""

#: musicbot/bot.py:5822
msgid "You must provide a section name and option name for this command."
msgstr ""

#: musicbot/bot.py:5838
#, python-format
msgid ""
"The section `%(section)s` is not available.\n"
"The available sections are:  %(sections)s"
msgstr ""

#: musicbot/bot.py:5846
#, python-format
msgid "The option `%(option)s` is not available."
msgstr ""

#: musicbot/bot.py:5874
#, python-format
msgid "Option `%(option)s` is not editable. Cannot save to disk."
msgstr ""

#: musicbot/bot.py:5883
#, python-format
msgid "Failed to save the option:  `%(option)s`"
msgstr ""

#: musicbot/bot.py:5898
#, python-format
msgid "Option `%(option)s` is not editable, value cannot be displayed."
msgstr ""

#: musicbot/bot.py:5921
#, python-format
msgid "Option `%(option)s` is not editable. Cannot update setting."
msgstr ""

#: musicbot/bot.py:5927 musicbot/bot.py:6851
msgid "You must provide a section, option, and value for this sub command."
msgstr ""

#: musicbot/bot.py:5931
#, python-format
msgid "Doing set with on %(config)s == %(value)s"
msgstr ""

#: musicbot/bot.py:5938
#, python-format
msgid "Option `%(option)s` was not updated!"
msgstr ""

#: musicbot/bot.py:5954
#, python-format
msgid "Option `%(option)s` is not editable. Cannot reset to default."
msgstr ""

#: musicbot/bot.py:5966
#, python-format
msgid "Resetting %(config)s to default %(value)s"
msgstr ""

#: musicbot/bot.py:5973
#, python-format
msgid "Option `%(option)s` was not reset to default!"
msgstr ""

#: musicbot/bot.py:6006
msgid "The option command is deprecated, use the config command instead."
msgstr ""

#: musicbot/bot.py:6026
msgid "Invalid option specified, use: info, update, or clear"
msgstr ""

#: musicbot/bot.py:6083
msgid "**Failed** to delete cache, check logs for more info..."
msgstr ""

#: musicbot/bot.py:6118
msgid "Queue page argument must be a whole number."
msgstr ""

#: musicbot/bot.py:6125
msgid "There are no songs queued! Queue something with a play command."
msgstr ""

#: musicbot/bot.py:6132
#, python-format
msgid ""
"Requested page number is out of bounds.\n"
"There are **%(total)s** pages."
msgstr ""

#: musicbot/bot.py:6175
msgid "Skipped the current playlist entry."
msgstr ""

#: musicbot/bot.py:6218
msgid "Not enough entries to paginate the queue."
msgstr ""

#: musicbot/bot.py:6222
msgid "Could not post queue message, no message to add reactions to."
msgstr ""

#: musicbot/bot.py:6224
msgid ""
"Try that again. MusicBot couldn't make or get a reference to the queue message.\n"
"If the issue persists, file a bug report."
msgstr ""

#: musicbot/bot.py:6334
msgid "Cannot use purge on private DM channel."
msgstr ""

#: musicbot/bot.py:6366
msgid "The given URL was not a valid URL."
msgstr ""

#: musicbot/bot.py:6376
#, python-format
msgid ""
"Could not extract info from input url\n"
"%(raw_error)s\n"
msgstr ""

#: musicbot/bot.py:6381
msgid "This does not seem to be a playlist."
msgstr ""

#: musicbot/bot.py:6551
msgid "Could not determine the discord User.  Try again."
msgstr ""

#: musicbot/bot.py:6622
msgid "Permissions cannot use channel and user mentions at the same time."
msgstr ""

#: musicbot/bot.py:6694
msgid "You must provide a group or option name for this command."
msgstr ""

#: musicbot/bot.py:6698
msgid "You must provide a group, option, and value to set for this command."
msgstr ""

#: musicbot/bot.py:6712
#, python-format
msgid "The %(option)s sub-command requires a group and permission name."
msgstr ""

#: musicbot/bot.py:6724
#, python-format
msgid ""
"The group `%(group)s` is not available.\n"
"The available groups are:  %(sections)s"
msgstr ""

#: musicbot/bot.py:6735
#, python-format
msgid "The permission `%(option)s` is not available."
msgstr ""

#: musicbot/bot.py:6771
#, python-format
msgid "Cannot add group `%(group)s` it already exists."
msgstr ""

#: musicbot/bot.py:6789
msgid "Cannot remove built-in group."
msgstr ""

#: musicbot/bot.py:6807 musicbot/bot.py:6846
msgid "The owner group is not editable."
msgstr ""

#: musicbot/bot.py:6815
#, python-format
msgid "Failed to save the group:  `%(group)s`"
msgstr ""

#: musicbot/bot.py:6855
#, python-format
msgid "Doing set on %(option)s with value: %(value)s"
msgstr ""

#: musicbot/bot.py:6862
#, python-format
msgid "Permission `%(option)s` was not updated!"
msgstr ""

#: musicbot/bot.py:6903
msgid ""
"Failed to change username. Did you change names too many times?\n"
"Remember name changes are limited to twice per hour.\n"
msgstr ""

#: musicbot/bot.py:6909
#, python-format
msgid ""
"Failed to change username due to error:  \n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:6931
msgid "Unable to change nickname: no permission."
msgstr ""

#: musicbot/bot.py:6939
#, python-format
msgid ""
"Failed to set nickname due to error:  \n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:6979
msgid "Custom emoji must be from this server to use as a prefix."
msgstr ""

#: musicbot/bot.py:6996
msgid ""
"Prefix per server is not enabled!\n"
"Use the config command to update the prefix instead."
msgstr ""

#: musicbot/bot.py:7097
msgid "You must provide a URL or attach a file."
msgstr ""

#: musicbot/bot.py:7107
#, python-format
msgid ""
"Unable to change avatar due to error:  \n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:7132
#, python-format
msgid "MusicBot found a %s with no guild!  This could be a problem."
msgstr ""

#: musicbot/bot.py:7144
#, python-format
msgid "Not currently connected to server `%(guild)s`"
msgstr ""

#: musicbot/bot.py:7185
msgid ""
"Invalid option given, use one of:  soft, full, upgrade, uppip, or upgit"
msgstr ""

#: musicbot/bot.py:7294
msgid "You must provide an ID or name."
msgstr ""

#: musicbot/bot.py:7305
#, python-format
msgid "No guild was found with the ID or name `%(input)s`"
msgstr ""

#: musicbot/bot.py:7362
#, python-format
msgid "Activating debug breakpoint ID: %(uuid)s"
msgstr ""

#: musicbot/bot.py:7402
msgid "Could not import `objgraph`, is it installed?"
msgstr ""

#: musicbot/bot.py:7477
msgid "Debug code ran with eval()."
msgstr ""

#: musicbot/bot.py:7487
msgid "Debug code ran with exec()."
msgstr ""

#: musicbot/bot.py:7490
msgid "Debug code failed to execute."
msgstr ""

#: musicbot/bot.py:7492
#, python-format
msgid ""
"Failed to execute debug code:\n"
"%(py_code)s\n"
"Exception: ```\n"
"%(ex_name)s:\n"
"%(ex_text)s```"
msgstr ""

#: musicbot/bot.py:7527 musicbot/bot.py:7588
#, python-format
msgid "Sub-command must be one of: %(options)s"
msgstr ""

#: musicbot/bot.py:7627
msgid "Could not locate git executable."
msgstr ""

#: musicbot/bot.py:7671
msgid "Failed while checking for updates via git command."
msgstr ""

#: musicbot/bot.py:7700
msgid "Package missing meta in pip report."
msgstr ""

#: musicbot/bot.py:7714
msgid "Failed to get pip update status due to some error."
msgstr ""

#: musicbot/bot.py:7775
msgid "Got a strange voice client entry."
msgstr ""

#: musicbot/bot.py:7885
msgid "Cookies already enabled."
msgstr ""

#: musicbot/bot.py:7892
msgid "Cookies must be uploaded to be enabled. (Missing cookies file.)"
msgstr ""

#: musicbot/bot.py:7905
#, python-format
msgid "Could not enable cookies due to error:  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:7919
#, python-format
msgid ""
"Could not rename cookies file due to error:  %(raw_error)s\n"
"Cookies temporarily disabled and will be re-enabled on next restart."
msgstr ""

#: musicbot/bot.py:7928
msgid ""
"No attached uploads were found, try again while uploading a cookie file."
msgstr ""

#: musicbot/bot.py:7937
#, python-format
msgid "Could not remove old, disabled cookies file:  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:7946
#, python-format
msgid "Error downloading the cookies file from discord:  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:7951
#, python-format
msgid "Could not save cookies to disk:  %(raw_error)s"
msgstr ""

#: musicbot/bot.py:7969
#, python-format
msgid "Got a message with no channel, somehow:  %s"
msgstr ""

#: musicbot/bot.py:7997
#, python-format
msgid "Ignoring command from myself (%s)"
msgstr ""

#: musicbot/bot.py:8005
#, python-format
msgid "Ignoring command from other bot (%s)"
msgstr ""

#: musicbot/bot.py:8013
#, python-format
msgid "Ignoring command from channel of type:  %s"
msgstr ""

#: musicbot/bot.py:8097
#, python-format
msgid "User in block list: %(id)s/%(name)s  tried command: %(command)s"
msgstr ""

#: musicbot/bot.py:8104
#, python-format
msgid "Message from %(id)s/%(name)s: %(message)s"
msgstr ""

#: musicbot/bot.py:8138
#, python-format
msgid "This command is not allowed for your permissions group:  %(group)s"
msgstr ""

#: musicbot/bot.py:8173
msgid "This command requires you to be in a Voice channel."
msgstr ""

#: musicbot/bot.py:8257
#, python-format
msgid "Invalid command usage, missing values for params: %(params)r"
msgstr ""

#: musicbot/bot.py:8299
#, python-format
msgid "Error in %(command)s: %(err_name)s: %(err_text)s"
msgstr ""

#: musicbot/bot.py:8323
#, python-format
msgid "Exception while handling command: %(command)s"
msgstr ""

#: musicbot/bot.py:8355
#, python-format
msgid "Cannot generate help for missing command:  %s"
msgstr ""

#: musicbot/bot.py:8359
#, python-format
msgid "Missing help data for command:  %s"
msgstr ""

#: musicbot/bot.py:8476
#, python-format
msgid "Leaving voice channel %s in %s due to inactivity."
msgstr ""

#: musicbot/bot.py:8485
msgid "MusicBot has become connected."
msgstr ""

#: musicbot/bot.py:8489
msgid "MusicBot has become disconnected."
msgstr ""

#: musicbot/bot.py:8494
#, python-format
msgid "Got a Socket Event:  %s"
msgstr ""

#: musicbot/bot.py:8510
msgid "VoiceState updated before on_ready finished"
msgstr ""

#: musicbot/bot.py:8523
#, python-format
msgid "Ignoring %s in %s as it is a bound voice channel."
msgstr ""

#: musicbot/bot.py:8532
#, python-format
msgid "%s has been detected as empty. Handling timeouts."
msgstr ""

#: musicbot/bot.py:8543
#, python-format
msgid "A user joined %s, cancelling timer."
msgstr ""

#: musicbot/bot.py:8556
#, python-format
msgid ""
"The bot got moved and the voice channel %s is empty. Handling timeouts."
msgstr ""

#: musicbot/bot.py:8565
#, python-format
msgid "The bot got moved and the voice channel %s is not empty."
msgstr ""

#: musicbot/bot.py:8614
#, python-format
msgid "No longer following user %s"
msgstr ""

#: musicbot/bot.py:8636
#, python-format
msgid "Following user `%(user)s` to channel:  %(channel)s"
msgstr ""

#: musicbot/bot.py:8656
msgid "VoiceState disconnect before.channel is None."
msgstr ""

#: musicbot/bot.py:8683
#, python-format
msgid ""
"Disconnected from voice by Discord API in: %(guild)s/%(channel)s (Code: "
"%(code)s) [S:%(state)s]"
msgstr ""

#: musicbot/bot.py:8705
#, python-format
msgid "Cannot use auto-join channel with type: %(type)s  in guild:  %(guild)s"
msgstr ""

#: musicbot/bot.py:8712
#, python-format
msgid "Cannot find the auto-joined channel, was it deleted?  Guild:  %s"
msgstr ""

#: musicbot/bot.py:8718
#, python-format
msgid "Reconnecting to auto-joined guild on channel:  %s"
msgstr ""

#: musicbot/bot.py:8731
#, python-format
msgid "Cannot auto join channel:  %s"
msgstr ""

#: musicbot/bot.py:8747
#, python-format
msgid "Bot has been added to guild: %s"
msgstr ""

#: musicbot/bot.py:8759
#, python-format
msgid "Left guild '%s' due to bot owner not found."
msgstr ""

#: musicbot/bot.py:8773
#, python-format
msgid "Creating data folder for guild %s"
msgstr ""

#: musicbot/bot.py:8781
#, python-format
msgid "Bot has been removed from guild: %s"
msgstr ""

#: musicbot/bot.py:8782
msgid "Updated guild list:"
msgstr ""

#: musicbot/bot.py:8797
#, python-format
msgid "Guild \"%s\" has become available."
msgstr ""

#: musicbot/bot.py:8803
#, python-format
msgid "Resuming player in \"%s\" due to availability."
msgstr ""

#: musicbot/bot.py:8820
#, python-format
msgid "Guild \"%s\" has become unavailable."
msgstr ""

#: musicbot/bot.py:8826
#, python-format
msgid "Pausing player in \"%s\" due to unavailability."
msgstr ""

#: musicbot/bot.py:8842
#, python-format
msgid "Guild update for:  %s"
msgstr ""

#: musicbot/bot.py:8899
#, python-format
msgid "Channel update for:  %(channel)s  --  %(changes)s"
msgstr ""

#: musicbot/config.py:87
#, python-format
msgid "Creating %s"
msgstr ""

#: musicbot/config.py:108
#, python-format
msgid "Loading config from:  %s"
msgstr ""

#: musicbot/config.py:1157
#, python-format
msgid ""
"Cannot store more than %s log files. Option LogsMaxKept will be limited "
"instead."
msgstr ""

#: musicbot/config.py:1165
msgid ""
"Config option LogsDateFormat is empty and this will break log file rotation."
" Using default instead."
msgstr ""

#: musicbot/config.py:1176
msgid ""
"Error while validating config options.\n"
"\n"
"Problem:\n"
"  Config option AudioCachePath is not a directory.\n"
"\n"
"Solution:\n"
"  Make sure the path you configured is a path to a folder / directory."
msgstr ""

#: musicbot/config.py:1193
msgid "An exception was thrown while validating AudioCachePath."
msgstr ""

#: musicbot/config.py:1197
#, python-format
msgid ""
"Error while validating config options.\n"
"\n"
"Problem:\n"
"  AudioCachePath config option could not be set due to an error:\n"
"  %(raw_error)s\n"
"\n"
"Solution:\n"
"  Double check the setting is a valid, accessible directory path."
msgstr ""

#: musicbot/config.py:1209
#, python-format
msgid "Audio Cache will be stored in:  %s"
msgstr ""

#: musicbot/config.py:1220
#, python-format
msgid ""
"Error while reading config options.\n"
"\n"
"Problem:\n"
"  No bot Token was specified in the config options or environment.\n"
"\n"
"Solution:\n"
"  Set the Token config option or set environment variable %(env_var)s with an App token."
msgstr ""

#: musicbot/config.py:1241
msgid ""
"StatusMessage config option is too long, it will be limited to 128 "
"characters."
msgstr ""

#: musicbot/config.py:1250
#, python-format
msgid ""
"The default playback speed must be between 0.5 and 100.0. The option value "
"of %.3f will be limited instead."
msgstr ""

#: musicbot/config.py:1260
msgid ""
"Cookies TXT file detected. MusicBot will pass them to yt-dlp.\n"
"Cookies are not recommended, may not be supported, and may totally break.\n"
"Copying cookies from your web-browser risks exposing personal data and \n"
"in the best case can result in your accounts being banned!\n"
"\n"
"You have been warned!  Good Luck!  \\U0001F596\n"
msgstr ""

#: musicbot/config.py:1278
msgid "Validating options with service data..."
msgstr ""

#: musicbot/config.py:1300
msgid "MusicBot does not have a user instance, cannot proceed."
msgstr ""

#: musicbot/config.py:1330
msgid "Config options file not found. Checking for alternatives..."
msgstr ""

#: musicbot/config.py:1343
#, python-format
msgid ""
"Renaming %(ini_file)s to %(option_file)s, you should probably turn file "
"extensions on."
msgstr ""

#: musicbot/config.py:1351
#, python-format
msgid "Copying existing example options file:  %(example_file)s"
msgstr ""

#: musicbot/config.py:1365
msgid "Something went wrong while trying to find a config option file."
msgstr ""

#: musicbot/config.py:1369
#, python-format
msgid ""
"Error locating config.\n"
"\n"
"Problem:\n"
"  Could not find or create a config file due to an error:\n"
"  %(raw_error)s\n"
"\n"
"Solution:\n"
"  Verify the config folder and files exist and can be read by MusicBot."
msgstr ""

#: musicbot/config.py:1387
#, python-format
msgid ""
"Error loading config.\n"
"\n"
"Problem:\n"
"  MusicBot could not read config file due to an error:\n"
"  %(raw_error)s\n"
"\n"
"Solution:\n"
"  Make sure the file is accessible and error free.\n"
"  Copy the example file from the repo if all else fails."
msgstr ""

#: musicbot/config.py:1411
msgid "Dev Bug! Config option has getter that is not available."
msgstr ""

#: musicbot/config.py:1416
msgid ""
"Dev Bug! Config option has invalid type, getter and default must be the same"
" type."
msgstr ""

#: musicbot/config.py:1436
msgid "Option was missing previously."
msgstr ""

#: musicbot/config.py:1451
#, python-format
msgid "Config section not in parsed config! Missing: %s"
msgstr ""

#: musicbot/config.py:1456
#, python-format
msgid "Saved config option: %(config)s  =  %(value)s"
msgstr ""

#: musicbot/config.py:1469
#, python-format
msgid "Failed to save config:  %s"
msgstr ""

#: musicbot/config.py:1988
msgid ""
"Option names are not unique between INI sections!  Resolver is disabled."
msgstr ""

#: musicbot/config.py:2165 musicbot/permissions.py:905
#, python-format
msgid "Failed to save default INI file at:  %s"
msgstr ""

#: musicbot/config.py:2255
#, python-format
msgid ""
"Error loading config value.\n"
"\n"
"Problem:\n"
"  The owner ID in [%(section)s] > %(option)s is not valid.\n"
"\n"
"Solution:\n"
"  Set %(option)s to a numerical ID or set it to `auto` or `0` for automatic owner binding."
msgstr ""

#: musicbot/config.py:2291
#, python-format
msgid ""
"Error loading config value.\n"
"\n"
"Problem:\n"
"  The config option [%(section)s] > %(option)s is not a valid file location.\n"
"\n"
"Solution:\n"
"  Check the path setting and make sure the file exists and is accessible to MusicBot."
msgstr ""

#: musicbot/config.py:2321
#, python-format
msgid ""
"Error loading config value.\n"
"\n"
"Problem:\n"
"  One of the IDs in option [%(section)s] > %(option)s is invalid.\n"
"\n"
"Solution:\n"
"  Ensure all IDs are numerical, and separated only by spaces or commas."
msgstr ""

#: musicbot/config.py:2357
#, python-format
msgid ""
"Invalid DebugLevel option \"%(value)s\" given, falling back to level: "
"%(fallback)s"
msgstr ""

#: musicbot/config.py:2378
#, python-format
msgid ""
"Option [%(section)s] > %(option)s has invalid config value '%(value)s' using"
" default instead."
msgstr ""

#: musicbot/config.py:2427
#, python-format
msgid ""
"Option [%(section)s] > %(option)s has a value greater than 100 %% "
"(%(value)s) and will be set to %(fallback)s instead."
msgstr ""

#: musicbot/config.py:2511
#, python-format
msgid ""
"Renaming INI file entry [%(old_s)s] > %(old_o)s  to  [%(new_s)s] > %(new_o)s"
msgstr ""

#: musicbot/config.py:2567
msgid "Upgrading config file with renamed options..."
msgstr ""

#: musicbot/config.py:2584
msgid "Failed to upgrade config.  You'll need to upgrade it manually."
msgstr ""

#: musicbot/config.py:2646
#, python-format
msgid "Could not load block list from file:  %s"
msgstr ""

#: musicbot/config.py:2755
#, python-format
msgid "We found a legacy blacklist file, it will be renamed to:  %s"
msgstr ""

#: musicbot/constructs.py:222
msgid ""
"Cannot load data for guild with ID 0. This is likely a bug in the code!"
msgstr ""

#: musicbot/constructs.py:231
#, python-format
msgid "No file for guild %(id)s/%(name)s"
msgstr ""

#: musicbot/constructs.py:240
#, python-format
msgid "Loading guild data for guild with ID:  %(id)s/%(name)s"
msgstr ""

#: musicbot/constructs.py:247
#, python-format
msgid "An OS error prevented reading guild data file:  %s"
msgstr ""

#: musicbot/constructs.py:258
#, python-format
msgid "Guild %(id)s/%(name)s has custom command prefix: %(prefix)s"
msgstr ""

#: musicbot/constructs.py:278
msgid ""
"Cannot save data for guild with ID 0. This is likely a bug in the code!"
msgstr ""

#: musicbot/constructs.py:302
msgid "Could not save guild specific data due to OS Error."
msgstr ""

#: musicbot/constructs.py:305
msgid "Failed to serialize guild specific data due to invalid data."
msgstr ""

#: musicbot/downloader.py:94
#, python-format
msgid "Forcing YTDLP to use User Agent:  %s"
msgstr ""

#: musicbot/downloader.py:105
#, python-format
msgid "MusicBot will use cookies for yt-dlp from:  %s"
msgstr ""

#: musicbot/downloader.py:111
msgid "Yt-dlp will use your configured proxy server."
msgstr ""

#: musicbot/downloader.py:134
msgid ""
"Original OAuth2 plugin is installed and will be used instead.\n"
"This may cause MusicBot to not close completely, or hang pending authorization!\n"
"To close MusicBot, you must manually Kill the MusicBot process!\n"
"Yt-dlp is being set to show warnings and other log messages, to show the Auth code.\n"
"Uninstall the yt-dlp-youtube-oauth2 package to use integrated OAuth2 features instead."
msgstr ""

#: musicbot/downloader.py:232
msgid "HEAD seems to have no headers..."
msgstr ""

#: musicbot/downloader.py:243
msgid "Checking media headers failed due to timeout."
msgstr ""

#: musicbot/downloader.py:246
#, python-format
msgid "Failed HEAD request for:  %s"
msgstr ""

#: musicbot/downloader.py:247
msgid "HEAD Request exception: "
msgstr ""

#: musicbot/downloader.py:315
#, python-format
msgid ""
"Sanitized YTDL Extraction Info (not JSON):\n"
"%s"
msgstr ""

#: musicbot/downloader.py:317
#, python-format
msgid "Sanitized YTDL Extraction Info (not JSON):  %s"
msgstr ""

#: musicbot/downloader.py:381
msgid "Song info extraction returned no data."
msgstr ""

#: musicbot/downloader.py:445
msgid "Cannot run extraction, loop is closed. (This is normal on shutdowns.)"
msgstr ""

#: musicbot/downloader.py:447
msgid "Cannot continue extraction, event loop is closed."
msgstr ""

#: musicbot/downloader.py:456
msgid "Spotify URL is invalid or not supported."
msgstr ""

#: musicbot/downloader.py:489 musicbot/downloader.py:510
#, python-format
msgid "Error in yt-dlp while downloading data: %(raw_error)s"
msgstr ""

#: musicbot/downloader.py:493
msgid "Download Error with stream URL"
msgstr ""

#: musicbot/downloader.py:498
msgid "Assuming content is a direct stream"
msgstr ""

#: musicbot/downloader.py:506
msgid "Cannot stream an invalid URL."
msgstr ""

#: musicbot/downloader.py:517
msgid ""
"Caught NoSupportingHandlers, trying again after replacing colon with space."
msgstr ""

#: musicbot/downloader.py:544
msgid ""
"Extractor youtube:search returned single-entry result, replacing base info "
"with entry info."
msgstr ""

#: musicbot/downloader.py:560
#, python-format
msgid "Called safe_extract_info with:  %(args)s, %(kws)s"
msgstr ""

#: musicbot/downloader.py:580
msgid "The local media file could not be found."
msgstr ""

#: musicbot/downloader.py:618
msgid "Missing __input_subject from YtdlpResponseDict"
msgstr ""

#: musicbot/downloader.py:623
msgid ""
"Entries is not a list in YtdlpResponseDict, set process=True to avoid this."
msgstr ""

#: musicbot/downloader.py:852
#, python-format
msgid "Warning, duration error for: %(url)s"
msgstr ""

#: musicbot/entry.py:46
msgid "module 'pymediainfo' not found, will fall back to ffprobe."
msgstr ""

#: musicbot/entry.py:159
#, python-format
msgid "Created future for %r"
msgstr ""

#: musicbot/entry.py:171
#, python-format
msgid "Completed futures for %(entry)r with %(callback)r"
msgstr ""

#: musicbot/entry.py:181
msgid "Unhandled exception in _for_each_future callback."
msgstr ""

#: musicbot/entry.py:223
#, python-format
msgid "Starting asyncio subprocess (%(process)s) with command: %(run)s"
msgstr ""

#: musicbot/entry.py:258
#, python-format
msgid ""
"Extraction did not provide a duration for this entry.\n"
"MusicBot cannot estimate queue times until it is downloaded.\n"
"Entry name:  %s"
msgstr ""

#: musicbot/entry.py:364 musicbot/entry.py:874 musicbot/entry.py:1084
msgid "Entry data is missing version number, cannot deserialize."
msgstr ""

#: musicbot/entry.py:367 musicbot/entry.py:877 musicbot/entry.py:1087
msgid "Entry data has the wrong version number, cannot deserialize."
msgstr ""

#: musicbot/entry.py:399
#, python-format
msgid "Deserialized URLPlaylistEntry has the wrong channel type:  %s"
msgstr ""

#: musicbot/entry.py:427
msgid ""
"Deserialized URLPlaylistEntry has an author ID but no channel for lookup!"
msgstr ""

#: musicbot/entry.py:437 musicbot/entry.py:943 musicbot/entry.py:1157
#, python-format
msgid "Could not load %s"
msgstr ""

#: musicbot/entry.py:486 musicbot/entry.py:948 musicbot/entry.py:1190
#, python-format
msgid "Getting ready for entry:  %r"
msgstr ""

#: musicbot/entry.py:503
msgid "Download cached with different extension..."
msgstr ""

#: musicbot/entry.py:512
msgid "Local size different from remote size. Re-downloading..."
msgstr ""

#: musicbot/entry.py:516
#, python-format
msgid "Download already cached at:  %s"
msgstr ""

#: musicbot/entry.py:536 musicbot/entry.py:1206
#, python-format
msgid ""
"MusicBot could not get duration data for this entry.\n"
"Queue time estimation may be unavailable until this track is cleared.\n"
"Entry file: %s"
msgstr ""

#: musicbot/entry.py:543
#, python-format
msgid "Got duration of %(time)s seconds for file:  %(file)s"
msgstr ""

#: musicbot/entry.py:566 musicbot/entry.py:1237
msgid "Exception while checking entry data."
msgstr ""

#: musicbot/entry.py:577 musicbot/entry.py:1248
#, python-format
msgid "Trying to get duration via pymediainfo for:  %s"
msgstr ""

#: musicbot/entry.py:583 musicbot/entry.py:1254
msgid "Failed to get duration via pymediainfo."
msgstr ""

#: musicbot/entry.py:590 musicbot/entry.py:1261
#, python-format
msgid "Trying to get duration via ffprobe for:  %s"
msgstr ""

#: musicbot/entry.py:593 musicbot/entry.py:1264
msgid "Could not locate ffprobe in your path!"
msgstr ""

#: musicbot/entry.py:614 musicbot/entry.py:1285
msgid "ffprobe returned something that could not be used."
msgstr ""

#: musicbot/entry.py:617 musicbot/entry.py:1288
msgid "ffprobe could not be executed for some reason."
msgstr ""

#: musicbot/entry.py:627 musicbot/entry.py:1298
#, python-format
msgid "Calculating mean volume of:  %s"
msgstr ""

#: musicbot/entry.py:630 musicbot/entry.py:1301
msgid "Could not locate ffmpeg on your path!"
msgstr ""

#: musicbot/entry.py:705
#, python-format
msgid "Download started:  %r"
msgstr ""

#: musicbot/entry.py:710
#, python-format
msgid "Download attempt %s of 3..."
msgstr ""

#: musicbot/entry.py:722
#, python-format
msgid ""
"Download incomplete, retrying in %(time).1f seconds.  Reason: %(raw_error)s"
msgstr ""

#: musicbot/entry.py:730
#, python-format
msgid "Download failed, not retrying! Reason:  %(raw_error)s"
msgstr ""

#: musicbot/entry.py:735
#, python-format
msgid "Download did not complete due to an error: %(raw_error)s"
msgstr ""

#: musicbot/entry.py:741
#, python-format
msgid "Download failed due to a yt-dlp error: %(raw_error)s"
msgstr ""

#: musicbot/entry.py:747
msgid "Extraction encountered an unhandled exception."
msgstr ""

#: musicbot/entry.py:751
#, python-format
msgid "Download failed due to an unhandled exception: %(raw_error)s"
msgstr ""

#: musicbot/entry.py:756
#, python-format
msgid "Download failed:  %r"
msgstr ""

#: musicbot/entry.py:757
msgid "Failed to extract data for the requested media."
msgstr ""

#: musicbot/entry.py:759
#, python-format
msgid "Download complete:  %r"
msgstr ""

#: musicbot/entry.py:906
#, python-format
msgid "Deserialized StreamPlaylistEntry has the wrong channel type:  %s"
msgstr ""

#: musicbot/entry.py:934
msgid ""
"Deserialized StreamPlaylistEntry has an author ID but no channel for lookup!"
msgstr ""

#: musicbot/entry.py:1119
#, python-format
msgid "Deserialized LocalFilePlaylistEntry has the wrong channel type:  %s"
msgstr ""

#: musicbot/entry.py:1147
msgid ""
"Deserialized LocalFilePlaylistEntry has an author ID but no channel for "
"lookup!"
msgstr ""

#: musicbot/entry.py:1213
#, python-format
msgid "Got duration of %(seconds)s seconds for file:  %(file)s"
msgstr ""

#: musicbot/filecache.py:120
#, python-format
msgid "Failed to delete cache file:  %s"
msgstr ""

#: musicbot/filecache.py:131
msgid "Audio cache directory has been removed."
msgstr ""

#: musicbot/filecache.py:138
msgid "Audio cache directory could not be removed or renamed."
msgstr ""

#: musicbot/filecache.py:145
msgid "Audio cache directory could not be removed."
msgstr ""

#: musicbot/filecache.py:154
msgid "Audio cache has no limits set, nothing to delete."
msgstr ""

#: musicbot/filecache.py:217
#, python-format
msgid "Audio cache deleted %(number)s file(s), total of %(size)s removed."
msgstr ""

#: musicbot/filecache.py:225
#, python-format
msgid ""
"Audio cached retained %(number)s file(s) from autoplaylist, total of "
"%(size)s retained."
msgstr ""

#: musicbot/filecache.py:234
#, python-format
msgid "Audio cache is now %(size)s over %(number)s file(s)."
msgstr ""

#: musicbot/filecache.py:250
msgid "Audio cache directory is missing, nothing to delete."
msgstr ""

#: musicbot/filecache.py:269
msgid ""
"Audio cache file is from autoplaylist but marked as busted, ignoring it."
msgstr ""

#: musicbot/filecache.py:280
#, python-format
msgid "Cache level requires cleanup. %s"
msgstr ""

#: musicbot/i18n.py:290
#, python-format
msgid "Failed to load log translations for any of:  [%s]  in:  %s"
msgstr ""

#: musicbot/json.py:27
#, python-format
msgid "Error parsing %s as JSON"
msgstr ""

#: musicbot/json.py:36
#, python-format
msgid "Could not grab data from JSON key: %s"
msgstr ""

#: musicbot/logs.py:131
msgid "Skipping logger setup, already set up"
msgstr ""

#: musicbot/logs.py:283
#, python-format
msgid "Log level was previously set via override to: %s"
msgstr ""

#: musicbot/logs.py:292
#, python-format
msgid "Changing log level to:  %s"
msgstr ""

#: musicbot/permissions.py:120
#, python-format
msgid "Permissions file not found, copying from:  %s"
msgstr ""

#: musicbot/permissions.py:129
#, python-format
msgid "Error copying example permissions file:  %s"
msgstr ""

#: musicbot/permissions.py:212
msgid "Validating permissions..."
msgstr ""

#: musicbot/permissions.py:280
#, python-format
msgid "Deleting group from permissions file:  %s"
msgstr ""

#: musicbot/permissions.py:285
#, python-format
msgid "Adding new group to permissions file:  %s"
msgstr ""

#: musicbot/permissions.py:300
msgid "Saving permissions file now."
msgstr ""

#: musicbot/permissions.py:306
msgid "ConfigUpdater could not parse the permissions file!"
msgstr ""

#: musicbot/permissions.py:308
msgid "You have a duplicate section, fix your Permissions file!"
msgstr ""

#: musicbot/permissions.py:310
#, python-format
msgid "Failed to save permissions group:  %s"
msgstr ""

#: musicbot/permissions.py:325
msgid "Dev Bug! Permission has getter that is not available."
msgstr ""

#: musicbot/permissions.py:330
msgid ""
"Dev Bug! Permission has invalid type, getter and default must be the same "
"type."
msgstr ""

#: musicbot/permissions.py:573
msgid "Max search items can't be larger than 100. Setting to 100."
msgstr ""

#: musicbot/permissions.py:656
#, python-format
msgid ""
"You do not have permission to play the requested media.\n"
"The yt-dlp extractor `%(extractor)s` is not permitted in your group."
msgstr ""

#: musicbot/player.py:86
#, python-format
msgid "Cleanup got called on the audio source:  %r"
msgstr ""

#: musicbot/player.py:191
#, python-format
msgid "MusicPlayer.skip() is called:  %s"
msgstr ""

#: musicbot/player.py:201
#, python-format
msgid "MusicPlayer.stop() is called:  %s"
msgstr ""

#: musicbot/player.py:216
msgid "Guild or network unavailable, cannot resume playback."
msgstr ""

#: musicbot/player.py:220
#, python-format
msgid "MusicPlayer.resume() is called:  %s"
msgstr ""

#: musicbot/player.py:240
#, python-format
msgid "MusicPlayer.pause() is called:  %s"
msgstr ""

#: musicbot/player.py:262
#, python-format
msgid "MusicPlayer.kill() is called:  %s"
msgstr ""

#: musicbot/player.py:288
msgid "Playback finished, but _current_entry is None."
msgstr ""

#: musicbot/player.py:343
msgid "Possible Warning from kill_current_player()"
msgstr ""

#: musicbot/player.py:362
#, python-format
msgid "MusicPlayer.play() is called:  %s"
msgstr ""

#: musicbot/player.py:372
msgid "MusicPlayer is dead, cannot play."
msgstr ""

#: musicbot/player.py:377
msgid "Guild or network unavailable, cannot start playback."
msgstr ""

#: musicbot/player.py:382
msgid "MusicPlayer was previously paused, resuming current player."
msgstr ""

#: musicbot/player.py:388
msgid "MusicPlayer already locked for playback, this call is ignored."
msgstr ""

#: musicbot/player.py:399
msgid "Failed to get next entry."
msgstr ""

#: musicbot/player.py:403
msgid "Failed to process entry for playback."
msgstr ""

#: musicbot/player.py:485
#, python-format
msgid "Creating player with options: ffmpeg %(before)s -i %(input)s %(after)s"
msgstr ""

#: musicbot/player.py:489
#, python-format
msgid "Playing %(source)r using %(client)r"
msgstr ""

#: musicbot/player.py:520
#, python-format
msgid "Skipping deletion of '%s', found song in queue"
msgstr ""

#: musicbot/player.py:524
#, python-format
msgid "Deleting file:  %s"
msgstr ""

#: musicbot/player.py:529
#, python-format
msgid "File deleted:  %s"
msgstr ""

#: musicbot/player.py:533
msgid "Cannot delete file, it is currently in use."
msgstr ""

#: musicbot/player.py:541
msgid "Cannot delete file, it was not found."
msgstr ""

#: musicbot/player.py:547
msgid "Error while trying to delete file."
msgstr ""

#: musicbot/player.py:623
msgid "Failed to deserialize player"
msgstr ""

#: musicbot/player.py:685
#, python-format
msgid "Data from ffmpeg: %s"
msgstr ""

#: musicbot/player.py:695
#, python-format
msgid "Error from ffmpeg: %s"
msgstr ""

#: musicbot/player.py:703
#, python-format
msgid "Warning from ffmpeg:  %s"
msgstr ""

#: musicbot/player.py:732
msgid "Unknown error decoding message from ffmpeg"
msgstr ""

#: musicbot/player.py:736
#, python-format
msgid "Decoded data from ffmpeg: %s"
msgstr ""

#: musicbot/playlist.py:121
#, python-format
msgid "Adding stream entry for URL:  %(url)s"
msgstr ""

#: musicbot/playlist.py:159
msgid "Could not extract information"
msgstr ""

#: musicbot/playlist.py:163
msgid "This is a playlist."
msgstr ""

#: musicbot/playlist.py:177
msgid "Entry info appears to be a stream, adding stream entry..."
msgstr ""

#: musicbot/playlist.py:203
msgid "Got text/html for content-type, this might be a stream."
msgstr ""

#: musicbot/playlist.py:210
#, python-format
msgid "Questionable content-type \"%(type)s\" for url:  %(url)s"
msgstr ""

#: musicbot/playlist.py:215
#, python-format
msgid "Adding URLPlaylistEntry for: %(subject)s"
msgstr ""

#: musicbot/playlist.py:235
#, python-format
msgid "Adding LocalFilePlaylistEntry for: %(subject)s"
msgstr ""

#: musicbot/playlist.py:280
#, python-format
msgid "Ignored video from compound playlist link with ID:  %s"
msgstr ""

#: musicbot/playlist.py:292
#, python-format
msgid ""
"Not allowing entry that is in song block list:  %(title)s  URL: %(url)s"
msgstr ""

#: musicbot/playlist.py:305
#, python-format
msgid ""
"Ignoring song in entries by '%s', duration longer than permitted maximum."
msgstr ""

#: musicbot/playlist.py:342
msgid "Could not add item"
msgstr ""

#: musicbot/playlist.py:343
#, python-format
msgid "Item: %s"
msgstr ""

#: musicbot/playlist.py:346
#, python-format
msgid "Skipped %s bad entries"
msgstr ""

#: musicbot/playlist.py:387
msgid "Reorder looping over entries."
msgstr ""

#: musicbot/playlist.py:469
#, python-format
msgid "Pre-downloading next track:  %r"
msgstr ""

#: musicbot/playlist.py:495
msgid "no duration data"
msgstr ""

#: musicbot/playlist.py:504
msgid "no duration data in current entry"
msgstr ""

#: musicbot/spotify.py:119
#, python-format
msgid "Invalid track_data, must be of type `track` got `%(type)s`"
msgstr ""

#: musicbot/spotify.py:301
#, python-format
msgid "Ignored non-track entry in playlist with type:  %s"
msgstr ""

#: musicbot/spotify.py:487
#, python-format
msgid "Spotify Album total tacks:  %(total)s  Next URL: %(url)s"
msgstr ""

#: musicbot/spotify.py:503
#, python-format
msgid ""
"Spotify Album Object may not be complete, expected %(total)s tracks but got "
"%(number)s"
msgstr ""

#: musicbot/spotify.py:507
msgid "Spotify Album has more tracks than initial total."
msgstr ""

#: musicbot/spotify.py:530
#, python-format
msgid "Spotify Playlist total tacks: %(total)s  Next URL: %(url)s"
msgstr ""

#: musicbot/spotify.py:546
#, python-format
msgid ""
"Spotify Playlist Object may not be complete, expected %(total)s tracks but "
"got %(number)s"
msgstr ""

#: musicbot/spotify.py:550
msgid "Spotify Playlist has more tracks than initial total."
msgstr ""

#: musicbot/spotify.py:555
#, python-format
msgid "Spotify Playlist contained %s usable tracks."
msgstr ""

#: musicbot/spotify.py:581 musicbot/spotify.py:617
#, python-format
msgid "Response status is not OK: [%(status)s] %(reason)s"
msgstr ""

#: musicbot/spotify.py:587 musicbot/spotify.py:623 musicbot/spotify.py:733
msgid "Response JSON did not decode to a dict!"
msgstr ""

#: musicbot/spotify.py:596
#, python-format
msgid "Failed making GET request to url:  %s"
msgstr ""

#: musicbot/spotify.py:602
#, python-format
msgid "Could not make GET to URL:  %(url)s  Reason:  %(raw_error)s"
msgstr ""

#: musicbot/spotify.py:632
#, python-format
msgid "Failed making POST request to url:  %s"
msgstr ""

#: musicbot/spotify.py:638
#, python-format
msgid "Could not make POST to URL:  %(url)s  Reason:  %(raw_error)s"
msgstr ""

#: musicbot/spotify.py:679
msgid "Created a new Guest Mode access token."
msgstr ""

#: musicbot/spotify.py:683
#, python-format
msgid ""
"API response did not contain the expected data. Missing key: %(raw_error)s"
msgstr ""

#: musicbot/spotify.py:689
#, python-format
msgid ""
"API response contained unexpected data.\n"
"%(raw_error)s"
msgstr ""

#: musicbot/spotify.py:696
msgid "Requested a token from Spotify, did not end up getting one"
msgstr ""

#: musicbot/spotify.py:700
msgid "Created a new Client Mode access token."
msgstr ""

#: musicbot/spotify.py:727
#, python-format
msgid "API response status is not OK: [%(status)s]  %(reason)s"
msgstr ""

#: musicbot/spotify.py:743
msgid "Failed to get Spotify Guest Token."
msgstr ""

#: musicbot/spotify.py:750
#, python-format
msgid "Failed to get Guest Token due to: %(raw_error)s"
msgstr ""

#: musicbot/utils.py:174
msgid "Only the owner can use this command."
msgstr ""

#: musicbot/utils.py:194
msgid "Only dev users can use this command."
msgstr ""

#: musicbot/utils.py:280 musicbot/utils.py:322
msgid "Cannot count members when voice_channel is None."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:85
#, python-format
msgid "Failed to save ytdlp oauth2 token data due to:  %s"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:102
#, python-format
msgid "Failed to load ytdlp oauth2 token data due to:  %s"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:140
msgid "Invalid cached OAuth2 token data."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:155
msgid "Access token expired, refreshing"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:212
#, python-format
msgid ""
"Failed to refresh OAuth2 access token due to:  %s\n"
"Restarting authorization flow..."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:230
msgid "Starting oauth2 flow..."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:249
#, python-format
msgid ""
"\n"
"NOTICE:\n"
"To give yt-dlp access to your account, visit:\n"
"  %s\n"
"Then enter this authorization code:  %s\n"
"You have %s seconds to complete authorization.\n"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:258
msgid ""
"The application may hang until authorization time out if closed at this "
"point. This is normal."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:264
msgid "Timed out while waiting for OAuth2 token."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:292
msgid "The device code has expired, restarting authorization flow for yt-dlp."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:298
msgid "Yt-dlp OAuth2 authorization successful."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:328
#, python-format
msgid "Adding OAuth2 Plugin to Yt-dlp IE:  %s"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:350
#, python-format
msgid "Default Yt-dlp Clients:  %s"
msgstr ""

#: run.py:105
msgid "Could not find git executable."
msgstr ""

#: run.py:154
msgid "Attempting to upgrade with `git pull` on current path."
msgstr ""

#: run.py:161
#, python-format
msgid "Result of git pull:  %s"
msgstr ""

#: run.py:169
msgid "Upgrade failed, you need to run `git pull` manually."
msgstr ""

#: run.py:179
msgid "Cannot execute pip."
msgstr ""

#: run.py:186
msgid "Error using -m method"
msgstr ""

#: run.py:232
msgid "PIP failed while calling sub-process."
msgstr ""

#: run.py:239
#, python-format
msgid "PIP failed due to missing Python executable?  (%s)"
msgstr ""

#: run.py:244
msgid "PIP failed due to OSError."
msgstr ""

#: run.py:265
msgid "Could not decode pip update report JSON."
msgstr ""

#: run.py:287
msgid "Cannot locate or execute python -m pip"
msgstr ""

#: run.py:290
#, python-format
msgid "Attempting to upgrade with `%s` on current path..."
msgstr ""

#: run.py:308
#, python-format
msgid ""
"Result of pip upgrade:\n"
"%s"
msgstr ""

#: run.py:313
#, python-format
msgid "Result exit code from pip upgrade: %s"
msgstr ""

#: run.py:328
msgid "Upgrade failed to execute or we could not understand the output"
msgstr ""

#: run.py:331
#, python-format
msgid "You may need to run `%s` manually."
msgstr ""

#: run.py:340
msgid "Press enter to continue . . ."
msgstr ""

#: run.py:353
msgid "Starting sanity checks"
msgstr ""

#: run.py:364
msgid "Required checks passed."
msgstr ""

#: run.py:377
msgid "Skipped checking for updates."
msgstr ""

#: run.py:379
msgid "Optional checks passed."
msgstr ""

#: run.py:387
msgid "Checking for Python 3.8+"
msgstr ""

#: run.py:391
#, python-format
msgid "Python 3.8+ is required. This version is %s"
msgstr ""

#: run.py:393
msgid "Attempting to locate Python 3.8..."
msgstr ""

#: run.py:401
msgid "Could not locate py.exe"
msgstr ""

#: run.py:412
msgid "Could not execute `py.exe -3.8` "
msgstr ""

#: run.py:416
msgid "Python 3 found.  Launching bot..."
msgstr ""

#: run.py:421
msgid "Trying \"python3.8\""
msgstr ""

#: run.py:424
msgid "Could not locate python3.8 on path."
msgstr ""

#: run.py:438
#, python-format
msgid ""
"\n"
"Python 3.8 found.  Re-launching bot using: %s run.py\n"
msgstr ""

#: run.py:443
msgid ""
"Could not find Python 3.8 or higher.  Please run the bot using Python 3.8"
msgstr ""

#: run.py:476
msgid "Ensuring we're in the right environment"
msgstr ""

#: run.py:499
#, python-format
msgid "Failed environment check, %s"
msgstr ""

#: run.py:540
#, python-format
msgid "Detected FFmpeg is installed at:  %s"
msgstr ""

#: run.py:542
msgid "Adding local bins/ folder environment PATH for bundled ffmpeg..."
msgstr ""

#: run.py:551
msgid ""
"MusicBot could not locate FFmpeg binary in your environment.\n"
"Please install FFmpeg so it is available in your environment PATH variable."
msgstr ""

#: run.py:556
msgid ""
"On Windows, you can add a pre-compiled EXE to the MusicBot `bin` folder,\n"
"or you can install FFmpeg system-wide using WinGet or by running the install.bat file."
msgstr ""

#: run.py:561
msgid ""
"On MacOS, you may be able to install FFmpeg via homebrew.\n"
"Otherwise, check the official FFmpeg site for build or install steps."
msgstr ""

#: run.py:566
msgid ""
"On Linux, many distros make FFmpeg available via system package managers.\n"
"Check for ffmpeg with your system package manager or build from sources."
msgstr ""

#: run.py:579
#, python-format
msgid "Less than %sMB of free space remains on this device"
msgstr ""

#: run.py:588
msgid ""
"\n"
"Checking for updates to MusicBot or dependencies..."
msgstr ""

#: run.py:605
msgid "No MusicBot updates available via `git` command."
msgstr ""

#: run.py:608
msgid ""
"Could not check for updates using `git` commands.  You should check "
"manually."
msgstr ""

#: run.py:615
msgid "The following packages can be updated:\n"
msgstr ""

#: run.py:621
#, python-format
msgid "  %s  to version:  %s\n"
msgstr ""

#: run.py:632
msgid "No dependency updates available via `pip` command."
msgstr ""

#: run.py:635
msgid ""
"Could not check for updates using `pip` commands.  You should check "
"manually."
msgstr ""

#: run.py:639
#, python-format
msgid ""
"You can run a guided update by using the command:\n"
"    %s ./update.py"
msgstr ""

#: run.py:657
msgid "Value is above the maximum limit."
msgstr ""

#: run.py:659
msgid "Value must not be negative."
msgstr ""

#: run.py:663
#, python-format
msgid "Value for Max Logs Kept must be a number from 0 to %s"
msgstr ""

#: run.py:673
#, python-format
msgid "Log level '%s' is not available."
msgstr ""

#: run.py:677
#, python-format
msgid "Log Level must be one of:  %s"
msgstr ""

#: run.py:687
msgid ""
"Launch a music playing discord bot built using discord.py, youtubeDL, and "
"ffmpeg."
msgstr ""

#: run.py:690
msgid "Available via Github:"
msgstr ""

#: run.py:694
msgid "For more help and support with this bot, join our discord:"
msgstr ""

#: run.py:696
msgid "This software is provided under the MIT License."
msgstr ""

#: run.py:698
msgid "See the `LICENSE` text file for complete details."
msgstr ""

#: run.py:710
msgid ""
"Override the default / system detected language for all text in MusicBot."
msgstr ""

#: run.py:719
msgid "Use this language for all server-side log messages from MusicBot."
msgstr ""

#: run.py:728
msgid ""
"Use this language for all messages sent to discord from MusicBot.\n"
"This does not prevent per-guild language selection."
msgstr ""

#: run.py:739
msgid "Print the MusicBot version information and exit."
msgstr ""

#: run.py:747
msgid "Skip all optional startup checks, including the update check."
msgstr ""

#: run.py:755
msgid "Skip only the disk space check at startup."
msgstr ""

#: run.py:763
msgid "Skip only the update check at startup."
msgstr ""

#: run.py:772
msgid ""
"Disable MusicBot from trying to install dependencies when it cannot import "
"them."
msgstr ""

#: run.py:783
#, python-format
msgid ""
"Specify how many log files to keep, between 0 and %s inclusive. (Default: "
"%s)"
msgstr ""

#: run.py:793
#, python-format
msgid "Override the log level settings set in config. Must be one of: %s"
msgstr ""

#: run.py:804
#, python-format
msgid ""
"Override the default date format used when rotating log files. This should "
"contain values compatible with strftime().  (Default:  '%s')"
msgstr ""

#: run.py:827
#, python-format
msgid "Version:  %s"
msgstr ""

#: run.py:902
msgid "Opened a new MusicBot instance. This terminal can be safely closed!"
msgstr ""

#: run.py:973
#, python-format
msgid "Loading MusicBot version:  %s"
msgstr ""

#: run.py:974
#, python-format
msgid "Log opened:  %s"
msgstr ""

#: run.py:975
#, python-format
msgid "Python version:  %s"
msgstr ""

#: run.py:984
#, python-format
msgid "Changing working directory to:  %s"
msgstr ""

#: run.py:988
msgid ""
"Cannot start the bot!  You started `run.py` in the wrong directory and we "
"could not locate `musicbot` and `.git` folders to verify a new directory "
"location."
msgstr ""

#: run.py:993
msgid ""
"For best results, start `run.py` from the same folder you cloned MusicBot into.\n"
"If you did not use git to clone the repository, you are strongly urged to."
msgstr ""

#: run.py:1047
msgid ""
"Certificate error is not a verification error, not trying certifi and "
"exiting."
msgstr ""

#: run.py:1049
msgid "Here is the exact error, it could be a bug."
msgstr ""

#: run.py:1065
msgid ""
"To easily add a certificate to Windows trust store, \n"
"you can open the failing site in Microsoft Edge or IE...\n"
msgstr ""

#: run.py:1071
msgid ""
"Could not get Issuer Certificate from default trust store, trying certifi "
"instead."
msgstr ""

#: run.py:1080
msgid "Syntax error (modification detected, did you edit the code?)"
msgstr ""

#: run.py:1083
msgid "Syntax error (this is a bug, not your fault)"
msgstr ""

#: run.py:1095
msgid ""
"Cannot start MusicBot due to an error!\n"
"\n"
"Problem:\n"
"  There was an error importing MusicBot or a dependency package.\n"
"\n"
"Solution:\n"
"  You need to manually install pip packages for MusicBot\n"
"  or launch without `--no-install-deps` and MusicBot will try to install them for you."
msgstr ""

#: run.py:1128
msgid "Attempting to install MusicBot dependency packages automatically...\n"
msgstr ""

#: run.py:1137
#, python-format
msgid ""
"MusicBot dependencies may not be installed!\n"
"\n"
"Problem:\n"
"  The pip install process ended with a possible error.\n"
"  Some or all of the the dependencies may be missing.\n"
"\n"
"Solution:\n"
"  You must manually install dependency packages.\n"
"  Open a CMD prompt / terminal to the MusicBot directory.\n"
"  You can try using the update scripts install packages.\n"
"  Or try this manual command:\n"
"    %(py_bin)s -m pip install -U -r ./requirements.txt\n"
"\n"
"You can also ask for help in MusicBot's support discord:\n"
"  https://discord.gg/bots"
msgstr ""

#: run.py:1158
msgid "OK, lets hope installing dependencies worked!"
msgstr ""

#: run.py:1170
msgid ""
"MusicBot got an ImportError after trying to install packages. MusicBot must "
"exit..."
msgstr ""

#: run.py:1172
msgid "The exception which caused the above error: "
msgstr ""

#: run.py:1191
msgid "MusicBot is doing a soft restart..."
msgstr ""

#: run.py:1195
msgid "MusicBot is doing a full process restart..."
msgstr ""

#: run.py:1201
msgid "Error starting bot"
msgstr ""

#: run.py:1206
msgid "Closing event loop."
msgstr ""

#: run.py:1211
#, python-format
msgid "Restarting in %s seconds..."
msgstr ""

#: run.py:1215
msgid "All done."
msgstr ""

#: run.py:1244
msgid "OK, we're closing!"
msgstr ""

#: musicbot/aliases.py:56
msgid ""
"Error while loading aliases.\n"
"\n"
"Problem:\n"
"  Your aliases files (aliases.json & example_aliases.json) are missing.\n"
"\n"
"Solution:\n"
"  Replace the alias config file(s) or copy them from:\n"
"    https://github.com/Just-Some-Bots/MusicBot/"
msgstr ""

#: musicbot/bot.py:853 musicbot/bot.py:8592
#, python-format
msgid "MusicBot is requesting to speak in channel: %s"
msgstr ""

#: musicbot/bot.py:982
msgid ""
"[BUG] MusicPlayer is missing a VoiceClient somehow.  You should probably "
"restart the bot."
msgstr ""

#: musicbot/bot.py:1190 musicbot/bot.py:4876
#, python-format
msgid "No thumbnail set for entry with URL: %s"
msgstr ""

#: musicbot/bot.py:1459
msgid "MusicBot needs to stop the auto playlist extraction and bail."
msgstr ""

#: musicbot/bot.py:1820
#, python-format
msgid "Cannot send non-response object:  %r"
msgstr ""

#: musicbot/bot.py:2157
msgid "MusicBot is now doing start up steps..."
msgstr ""

#: musicbot/bot.py:2164
msgid "Start up failed at login."
msgstr ""

#: musicbot/bot.py:2396
#, python-format
msgid "Cannot bind to non Messageable channel with ID:  %d"
msgstr ""

#: musicbot/bot.py:2445
#, python-format
msgid "Got None for auto join channel with ID:  %d"
msgstr ""

#: musicbot/bot.py:2451
#, python-format
msgid "Cannot auto join a Private/Non-Guild channel with ID:  %d"
msgstr ""

#: musicbot/bot.py:2459
#, python-format
msgid "Cannot auto join to non-connectable channel with ID:  %d"
msgstr ""

#: musicbot/bot.py:2475
msgid "Auto joining voice channels:"
msgstr ""

#: musicbot/bot.py:2483 musicbot/bot.py:2486
msgid "Not auto joining any voice channels"
msgstr ""

#: musicbot/bot.py:2629
#, python-format
msgid "    Delete Now Playing: %s"
msgstr ""

#: musicbot/bot.py:2728
#, python-format
msgid "The requested song `%(subject)s` is blocked by the song block list."
msgstr ""

#: musicbot/bot.py:3081
#, python-format
msgid "Not removing user from block list, not listed:  %(id)s/%(name)s"
msgstr ""

#: musicbot/bot.py:3181
msgid "Invalid sub-command given. Use `help blocksong` for usage examples."
msgstr ""

#: musicbot/bot.py:3429
#, python-format
msgid "No playlist file exists with the name: `%(playlist)s`"
msgstr ""

#: musicbot/bot.py:4229
msgid "Detected a Spotify URL, but Spotify is not enabled."
msgstr ""

#: musicbot/bot.py:4278
#, python-format
msgid "YouTube search returned no results for:  %(url)s"
msgstr ""

#: musicbot/bot.py:4335
msgid "Extracted an entry with 'youtube:playlist' as extractor key"
msgstr ""

#: musicbot/bot.py:5101
msgid ""
"There is nothing currently playing. Play something with a play command."
msgstr ""

#: musicbot/bot.py:5115
#, python-format
msgid ""
"Missing permissions to delete queue file for %(guild)s(%(id)s) at: %(path)s"
msgstr ""

#: musicbot/bot.py:5120
#, python-format
msgid "OS level error while trying to delete queue file: %(path)s"
msgstr ""

#: musicbot/bot.py:5533
msgid "The speed you provided is invalid. Use a number between 0.5 and 100."
msgstr ""

#: musicbot/bot.py:6544
msgid ""
"Invalid user ID or server nickname, please double-check the ID and try "
"again."
msgstr ""

#: musicbot/bot.py:6657
#, python-format
msgid ""
"Unable to reload Permissions due to an error:\n"
"%(raw_error)s"
msgstr ""

#: musicbot/bot.py:7026
msgid "This command can only be used in guilds."
msgstr ""

#: musicbot/bot.py:7031
msgid "Invalid sub-command given. Use the help command for more information."
msgstr ""

#: musicbot/bot.py:7052
#, python-format
msgid "Cannot set language to `%(locale)s` it is not available."
msgstr ""

#: musicbot/bot.py:8848
#, python-format
msgid "Guild attribute %(attr)s is now: %(new)s  -- Was: %(old)s"
msgstr ""

#: musicbot/config.py:1083
msgid "Generating new config options files..."
msgstr ""

#: musicbot/config.py:1093
#, python-format
msgid ""
"Error creating default config options file.\n"
"\n"
"Problem:\n"
"  MusicBot attempted to generate the config files but failed due to an error:\n"
"  %(raw_error)s\n"
"\n"
"Solution:\n"
"  Make sure MusicBot can read and write to your config files.\n"
msgstr ""

#: musicbot/config.py:1127
#, python-format
msgid ""
"Error while reading config.\n"
"\n"
"Problem:\n"
"  One or more required config option sections are missing.\n"
"  The missing sections are:\n"
"  %(sections)s\n"
"\n"
"Solution:\n"
"  Repair your config options file.\n"
"  Each [Section] must appear only once, with no other text on the same line.\n"
"  Each section must have at least one option.\n"
"  Use the example options as a template or copy it from the repository."
msgstr ""

#: musicbot/config.py:1284
msgid "Acquired owner ID via API"
msgstr ""

#: musicbot/config.py:1288
msgid ""
"Error while fetching 'OwnerID' automatically.\n"
"\n"
"Problem:\n"
"  Discord App info is not available.\n"
"  This could be a temporary API outage or a bug.\n"
"\n"
"Solution:\n"
"  Manually set the 'OwnerID' config option or try again later."
msgstr ""

#: musicbot/config.py:1306
msgid ""
"Error validating config options.\n"
"\n"
"Problem:\n"
"  The 'OwnerID' config is the same as your Bot / App ID.\n"
"\n"
"Solution:\n"
"  Do not use the Bot or App ID in the 'OwnerID' field."
msgstr ""

#: musicbot/config.py:1358
#, python-format
msgid ""
"Could not locate config options or example options files.\n"
"MusicBot will generate the config files at the location:\n"
"  %(cfg_file)s"
msgstr ""

#: musicbot/config.py:2625
#, python-format
msgid "Block list file not found:  %s"
msgstr ""

#: musicbot/config.py:2684 musicbot/config.py:2717
#, python-format
msgid "Could not update the block list file:  %s"
msgstr ""

#: musicbot/config.py:2744
#, python-format
msgid "Loaded User Block list with %s entries."
msgstr ""

#: musicbot/config.py:2801
#, python-format
msgid "Loaded a Song Block list with %s entries."
msgstr ""

#: musicbot/downloader.py:437
#, python-format
msgid "Called extract_info with:  '%(subject)s', %(args)s, %(kws)s"
msgstr ""

#: musicbot/entry.py:383
#, python-format
msgid "Deserialized URLPlaylistEntry cannot find channel with ID:  %s"
msgstr ""

#: musicbot/entry.py:421
#, python-format
msgid "Deserialized URLPlaylistEntry cannot find author with ID:  %s"
msgstr ""

#: musicbot/entry.py:474
#, python-format
msgid "Cannot download Spotify links, processing error with type: %(type)s"
msgstr ""

#: musicbot/entry.py:554 musicbot/entry.py:1224
msgid ""
"There as a problem with working out EQ, likely caused by a strange "
"installation of FFmpeg. This has not impacted the ability for the bot to "
"work, but will mean your tracks will not be equalized."
msgstr ""

#: musicbot/entry.py:656 musicbot/entry.py:1327
msgid "Could not parse 'I' in normalize json."
msgstr ""

#: musicbot/entry.py:664 musicbot/entry.py:1335
msgid "Could not parse 'LRA' in normalize json."
msgstr ""

#: musicbot/entry.py:672 musicbot/entry.py:1343
msgid "Could not parse 'TP' in normalize json."
msgstr ""

#: musicbot/entry.py:680 musicbot/entry.py:1351
msgid "Could not parse 'thresh' in normalize json."
msgstr ""

#: musicbot/entry.py:688 musicbot/entry.py:1359
msgid "Could not parse 'offset' in normalize json."
msgstr ""

#: musicbot/entry.py:890
#, python-format
msgid "Deserialized StreamPlaylistEntry cannot find channel with ID:  %s"
msgstr ""

#: musicbot/entry.py:928
#, python-format
msgid "Deserialized StreamPlaylistEntry cannot find author with ID:  %s"
msgstr ""

#: musicbot/entry.py:1103
#, python-format
msgid "Deserialized LocalFilePlaylistEntry cannot find channel with ID:  %s"
msgstr ""

#: musicbot/entry.py:1141
#, python-format
msgid "Deserialized LocalFilePlaylistEntry cannot find author with ID:  %s"
msgstr ""

#: musicbot/filecache.py:304
msgid "Auto playlist has no cache map, moving on."
msgstr ""

#: musicbot/filecache.py:312
#, python-format
msgid "Loaded auto playlist cache map with %s entries."
msgstr ""

#: musicbot/filecache.py:316
msgid "Failed to load auto playlist cache map."
msgstr ""

#: musicbot/filecache.py:332
msgid "Cachemap defer flag is set, not yet saving to disk..."
msgstr ""

#: musicbot/filecache.py:341
#, python-format
msgid "Saved auto playlist cache map with %s entries."
msgstr ""

#: musicbot/filecache.py:345
msgid "Failed to save auto playlist cache map."
msgstr ""

#: musicbot/filecache.py:363
#, python-format
msgid ""
"Auto playlist cache map conflict on Key: %(file)s  Old: %(old)s  New: "
"%(new)s"
msgstr ""

#: musicbot/i18n.py:229
#, python-format
msgid "Lang Argument Error:  %s"
msgstr ""

#: musicbot/i18n.py:338
#, python-format
msgid ""
"Failed to load discord translations for any of:  [%s]  guild:  %s  in:  %s"
msgstr ""

#: musicbot/json.py:14
#, python-format
msgid "Loading JSON file: %s"
msgstr ""

#: musicbot/logs.py:321
msgid "MusicBot loggers have been called to shut down."
msgstr ""

#: musicbot/permissions.py:136
#, python-format
msgid ""
"Could not locate config permissions or example permissions files.\n"
"MusicBot will generate the config files at the location:\n"
"  %(perms_file)s"
msgstr ""

#: musicbot/permissions.py:163
msgid "Generating new config permissions files..."
msgstr ""

#: musicbot/permissions.py:172
#, python-format
msgid ""
"Error creating default config permissions file.\n"
"\n"
"Problem:\n"
"  MusicBot attempted to generate the config files but failed due to an error:\n"
"  %(raw_error)s\n"
"\n"
"Solution:\n"
"  Make sure MusicBot can read and write to your config files.\n"
msgstr ""

#: musicbot/permissions.py:195
msgid "Config 'OwnerID' is set auto, will set correctly later."
msgstr ""

#: musicbot/permissions.py:214
msgid "Setting auto 'OwnerID' for owner permissions group."
msgstr ""

#: musicbot/permissions.py:274
#, python-format
msgid "Updating group in permissions file:  %s"
msgstr ""

#: musicbot/player.py:536
msgid "Cannot delete file due to a permission error."
msgstr ""

#: musicbot/player.py:552
msgid "Could not delete file, giving up and moving on"
msgstr ""

#: musicbot/player.py:618
#, python-format
msgid "Deserialize returned an object that is not a MusicPlayer:  %s"
msgstr ""

#: musicbot/playlist.py:195
#, python-format
msgid "Invalid content type `%(type)s` for URL: %(url)s"
msgstr ""

#: musicbot/playlist.py:317
#, python-format
msgid "Not adding YouTube video because it is marked private or deleted:  %s"
msgstr ""

#: musicbot/spotify.py:492
#, python-format
msgid "Getting Spotify Album Next URL:  %s"
msgstr ""

#: musicbot/spotify.py:535
#, python-format
msgid "Getting Spotify Playlist Next URL:  %s"
msgstr ""

#: musicbot/spotify.py:672
msgid ""
"Failed to get a guest token from Spotify, please try specifying client ID "
"and client secret"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:92
msgid "Loading YouTube TV OAuth2 token data."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:110
msgid "Storing YouTube TV OAuth2 token data"
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:176
msgid ""
"YouTube cookies have been provided, but OAuth2 is being used. If you "
"encounter problems, stop providing YouTube cookies to yt-dlp."
msgstr ""

#: musicbot/ytdlp_oauth2_plugin.py:194
msgid "Refreshing YouTube TV oauth2 token..."
msgstr ""

#: run.py:235
msgid "PIP failed due to Permission Error."
msgstr ""

#: run.py:523
#, python-format
msgid ""
"MusicBot could not write files in the following directory:\n"
"%(dir)s"
msgstr ""

#: run.py:527
msgid "Please make sure MusicBot can read and write in the above directory."
msgstr ""

#: run.py:534
msgid "Failed to clean up write-test path."
msgstr ""

#: run.py:817
msgid ""
"Supply a directory where MusicBot can store all mutable files.\n"
"Essentially treats the install directory as read-only.\n"
"MusicBot must have permission to create this directory.\n"
msgstr ""

#: musicbot/bot.py:2412
msgid "Private Channel"
msgstr ""

#: musicbot/bot.py:2414
msgid "Unknown User DM"
msgstr ""

#: musicbot/bot.py:2416
#, python-format
msgid "User DM: %(name)s"
msgstr ""

#: musicbot/bot.py:2420
msgid "Unknown Channel (Partial)"
msgstr ""

#: musicbot/bot.py:2422
#, python-format
msgid "Unnamed Channel: %(id)s"
msgstr ""

#: musicbot/bot.py:2751
msgid "Unknown Channel"
msgstr ""

#. TRANSLATORS: Placeholder for empty track title.
#: musicbot/entry.py:299 musicbot/entry.py:821 musicbot/entry.py:1020
msgid "Unknown"
msgstr ""

#: musicbot/bot.py:716
#, python-format
msgid "MusicBot may use up to %(rate).2f %(suffix)s of bandwidth."
msgstr ""

#: musicbot/bot.py:2699
#, python-format
msgid "  Audio Codec: %s"
msgstr ""

#: musicbot/player.py:446
#, python-format
msgid "Probed codec and bitrate: %(name)s at %(rate)s kbps"
msgstr ""

#~ msgid ""
#~ "Error while loading alises.\n"
#~ "\n"
#~ "Problem:\n"
#~ "  Your aliases files (aliases.json & example_aliases.json) are missing.\n"
#~ "\n"
#~ "Solution:\n"
#~ "  Replace the alias config file(s) or copy them from:\n"
#~ "    https://github.com/Just-Some-Bots/MusicBot/"
#~ msgstr ""

#~ msgid ""
#~ "[BUG] MusicPlayer is missing a VoiceClient some how.  You should probably "
#~ "restart the bot."
#~ msgstr ""

#, python-format
#~ msgid "No thumbnail set for entry with url: %s"
#~ msgstr ""

#~ msgid "MusicBot needs to stop the autoplaylist extraction and bail."
#~ msgstr ""

#, python-format
#~ msgid "Cannot send non-reponse object:  %r"
#~ msgstr ""

#, python-format
#~ msgid "Cannot bind to non-messagable channel with ID:  %d"
#~ msgstr ""

#, python-format
#~ msgid "Got None for autojoin channel with ID:  %d"
#~ msgstr ""

#, python-format
#~ msgid "Cannot autojoin a Private/Non-Guild channel with ID:  %d"
#~ msgstr ""

#, python-format
#~ msgid "Cannot autojoin to non-connectable channel with ID:  %d"
#~ msgstr ""

#~ msgid "Autojoining voice channels:"
#~ msgstr ""

#~ msgid "Not autojoining any voice channels"
#~ msgstr ""

#, python-format
#~ msgid "    Delete Nowplaying: %s"
#~ msgstr ""

#, python-brace-format
#~ msgid "The requested song `{subject}` is blocked by the song blocklist."
#~ msgstr ""

#, python-format
#~ msgid "Not removing user from blocklist, not listed:  %(id)s/%(name)s"
#~ msgstr ""

#~ msgid "Ivalid sub-command given. Use `help blocksong` for usage examples."
#~ msgstr ""

#~ msgid "Detected a spotify URL, but spotify is not enabled."
#~ msgstr ""

#, python-format
#~ msgid "Youtube search returned no results for:  %(url)s"
#~ msgstr ""

#~ msgid "Extracted an entry with youtube:playlist as extractor key"
#~ msgstr ""

#, python-format
#~ msgid ""
#~ "Failed to exctract info due to error:\n"
#~ "%(raw_error)s"
#~ msgstr ""

#~ msgid "The speed you proivded is invalid. Use a number between 0.5 and 100."
#~ msgstr ""

#~ msgid ""
#~ "Invalid user ID or server nickname, please double check the ID and try "
#~ "again."
#~ msgstr ""

#, python-format
#~ msgid ""
#~ "Unable to reload Permissions due to an errror:\n"
#~ "%(raw_error)s"
#~ msgstr ""

#, python-brace-format
#~ msgid "Guild attribute {name} is now: {a_val}  -- Was: {b_val}"
#~ msgstr ""

#, python-format
#~ msgid ""
#~ "Error while reading config.\n"
#~ "\n"
#~ "Problem:\n"
#~ "  One or more required config option sections are missing.\n"
#~ "  The missing sections are:\n"
#~ "  %(sections)s\n"
#~ "\n"
#~ "Solution:\n"
#~ "  Repair your config optoins file.\n"
#~ "  Each [Section] must appear only once, with no other text on the same line.\n"
#~ "  Each section must have at least one option.\n"
#~ "  Use the example options as a template or copy it from the repository."
#~ msgstr ""

#~ msgid "Acquired owner id via API"
#~ msgstr ""

#~ msgid ""
#~ "Error while fetching OwnerID automatically.\n"
#~ "\n"
#~ "Problem:\n"
#~ "  Discord App info is not available.\n"
#~ "  This could be a temporary API outage or a bug.\n"
#~ "\n"
#~ "Solution:\n"
#~ "  Manually set the OwnerID config option or try again later."
#~ msgstr ""

#~ msgid ""
#~ "Error validating config options.\n"
#~ "\n"
#~ "Problem:\n"
#~ "  The OwnerID config is the same as your Bot / App ID.\n"
#~ "\n"
#~ "Solution:\n"
#~ "  Do not use the Bot or App ID in the OwnerID field."
#~ msgstr ""

#, python-format
#~ msgid "Generated a new %(example_file)s and copied it to %(option_file)s"
#~ msgstr ""

#, python-format
#~ msgid "Blocklist file not found:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Could not update the blocklist file:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Loaded User Blocklist with %s entires."
#~ msgstr ""

#, python-format
#~ msgid "Loaded a Song Blocklist with %s entries."
#~ msgstr ""

#, python-brace-format
#~ msgid "Called extract_info with:  '{song_subject}', {args}, {kwargs}"
#~ msgstr ""

#, python-format
#~ msgid "Deserialized URLPlaylistEntry cannot find channel with id:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Deserialized URLPlaylistEntry cannot find author with id:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Cannot download spotify links, processing error with type: %(type)s"
#~ msgstr ""

#~ msgid ""
#~ "There as a problem with working out EQ, likely caused by a strange "
#~ "installation of FFmpeg. This has not impacted the ability for the bot to "
#~ "work, but will mean your tracks will not be equalised."
#~ msgstr ""

#~ msgid "Could not parse I in normalise json."
#~ msgstr ""

#~ msgid "Could not parse LRA in normalise json."
#~ msgstr ""

#~ msgid "Could not parse TP in normalise json."
#~ msgstr ""

#~ msgid "Could not parse thresh in normalise json."
#~ msgstr ""

#~ msgid "Could not parse offset in normalise json."
#~ msgstr ""

#, python-format
#~ msgid "Deserialized StreamPlaylistEntry cannot find channel with id:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Deserialized StreamPlaylistEntry cannot find author with id:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Deserialized LocalFilePlaylistEntry cannot find channel with id:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Deserialized LocalFilePlaylistEntry cannot find author with id:  %s"
#~ msgstr ""

#~ msgid "Autoplaylist has no cache map, moving on."
#~ msgstr ""

#, python-format
#~ msgid "Loaded autoplaylist cache map with %s entries."
#~ msgstr ""

#~ msgid "Failed to load autoplaylist cache map."
#~ msgstr ""

#, python-format
#~ msgid "Saved autoplaylist cache map with %s entries."
#~ msgstr ""

#~ msgid "Failed to save autoplaylist cache map."
#~ msgstr ""

#, python-format
#~ msgid ""
#~ "Autoplaylist cache map conflict on Key: %(file)s  Old: %(old)s  New: %(new)s"
#~ msgstr ""

#, python-format
#~ msgid "Lang Args Error:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Failed to load discord translations for any of:  [%s]  in:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Init JSON obj from file: %s"
#~ msgstr ""

#~ msgid "MusicBot loggers have been called to shutdown."
#~ msgstr ""

#~ msgid "OwnerID is set auto, will set correctly later."
#~ msgstr ""

#~ msgid "Setting auto OwnerID for owner permissions group."
#~ msgstr ""

#, python-format
#~ msgid "Updating group in permssions file:  %s"
#~ msgstr ""

#~ msgid "Cannot delete file due to a permissions error."
#~ msgstr ""

#~ msgid "[Config:SaveVideos] Could not delete file, giving up and moving on"
#~ msgstr ""

#, python-format
#~ msgid "Deserialize returned a non-MusicPlayer:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Invalid content type `%(type)s` for url: %(url)s"
#~ msgstr ""

#, python-format
#~ msgid "Not adding youtube video because it is marked private or deleted:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Getting Spofity Album Next URL:  %s"
#~ msgstr ""

#, python-format
#~ msgid "Getting Spofity Playlist Next URL:  %s"
#~ msgstr ""

#~ msgid ""
#~ "Failed to get a guest token from Spotify, please try specifying client id "
#~ "and client secret"
#~ msgstr ""

#~ msgid "Loading Youtube TV OAuth2 token data."
#~ msgstr ""

#~ msgid "Storing Youtube TV OAuth2 token data"
#~ msgstr ""

#~ msgid "validate ytdlp token..."
#~ msgstr ""

#~ msgid "init oauth for ytdlp"
#~ msgstr ""

#~ msgid "handling oauth2"
#~ msgstr ""

#~ msgid ""
#~ "Youtube cookies have been provided, but OAuth2 is being used. If you "
#~ "encounter problems, stop providing Youtube cookies to yt-dlp."
#~ msgstr ""

#~ msgid "refreshing oauth2 token"
#~ msgstr ""

#~ msgid "PIP failed due to Permissions Error."
#~ msgstr ""

#~ msgid "Current working directory does not seem to be writable"
#~ msgstr ""

#~ msgid "Please move the bot to a folder that is writable"
#~ msgstr ""
